<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning | Nature Biotechnology</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nbt.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"classification-and-taxonomy;machine-learning","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Biotechnology","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41587-026-03005-7"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Pau Piera Líndez","Lasse Schnell Danielsen","Iva Kovačić","Marc Pielies Avellí","Joseph Nesme","Lars Juhl Jensen","Jakob Nybo Andersen","Søren Johannes Sørensen","Simon Rasmussen"],"publishedAt":1770163200,"publishedAtString":"2026-02-04","title":"Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Classification and taxonomy,Machine learning"},"journal":{"pcode":"nbt","title":"nature biotechnology","volume":null,"issue":null,"id":41587,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"download-collection-test","active":false},{"name":"download-issue-test","active":false},{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"CN","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://sgtm.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2,h2.app-access-wall__title{-webkit-font-smoothing:antialiased}.c-card--major .c-card__title,.u-h2,h2{font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.app-cta-group{display:flex;flex-direction:column;gap:0 16px;width:100%}@media only screen and (min-width:540px){.app-cta-group{align-items:baseline;flex-direction:row}}.app-cta-group>*{flex:1 1 auto}@media only screen and (min-width:540px){.app-cta-group>*{flex:0 1 auto}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box svg+.c-article__button-text{margin-left:8px}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}p{overflow-wrap:break-word;word-break:break-word}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;padding:0}.c-article-identifiers__item{list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #fff;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#0067c5;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex:1 1 auto;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:24px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__container{margin:0 auto;max-width:1280px;padding:0 16px}.c-context-bar__title{display:none}.app-researcher-popup__link.hover,.app-researcher-popup__link.visited,.app-researcher-popup__link:hover,.app-researcher-popup__link:visited,.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-authors-search__list{align-items:center;display:flex;flex-wrap:wrap;gap:16px 16px;justify-content:center}@media only screen and (min-width:320px){.c-article-authors-search__list{justify-content:normal}}.c-article-authors-search__text{align-items:center;display:flex;flex-flow:column wrap;font-size:14px;justify-content:center}@media only screen and (min-width:320px){.c-article-authors-search__text{flex-direction:row;font-size:16px}}.c-article-authors-search__links-text{font-weight:700;margin-right:8px;text-align:center}@media only screen and (min-width:320px){.c-article-authors-search__links-text{text-align:left}}.c-article-authors-search__list-item--left{flex:1 1 100%}@media only screen and (min-width:320px){.c-article-authors-search__list-item--left{flex-basis:auto}}.c-article-authors-search__list-item--right{flex:1 1 auto}.c-article-identifiers{margin:0}.c-article-identifiers__item{border-right:2px solid #cedbe0;color:#222;font-size:14px}@media only screen and (min-width:320px){.c-article-identifiers__item{font-size:16px}}.c-article-identifiers__item:last-child{border-right:none}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}.app-author-list{color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;line-height:1.4;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-weight:400;line-height:1.5;margin-bottom:4px}.c-ad__label,.c-meta{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem}.c-meta{color:inherit;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{display:inline-block;fill:currentcolor;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}@media print{.c-header__menu{display:none}}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -4px;padding:0}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__show-text-sm{display:inline;visibility:visible}@media only screen and (min-width:540px){.c-header__show-text-sm{display:none;visibility:hidden}.c-header__show-text-sm:first-child+*{margin-block-start:0}}@media print{.c-header__dropdown{display:none}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border-radius:2px;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button svg,.u-button--primary svg{fill:currentcolor}.u-button{border:1px solid #069;color:#069}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{display:inline-block;fill:currentcolor;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.u-mb-48{margin-bottom:48px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-f88185239c.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-f88185239c.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-fb7cb72232.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">



<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>


<script>
    (function (w, d, s) {
        var urlParams = new URLSearchParams(w.location.search);
        if (urlParams.get('gptAdsTest') !== null) {
            d.addEventListener('sncc:initialise', function (e) {
                var t = d.createElement(s);
                var h = d.getElementsByTagName(s)[0];
                t.src = 'https://' + (e.detail.C03 ? 'securepubads.g.doubleclick' : 'pagead2.googlesyndication') + '.net/tag/js/gpt.js';
                t.async = false;
                t.onload = function () {
                    var n = d.createElement(s);
                    n.src = 'https://fed-libs.nature.com/production/gpt-ads-gtm.min.js';
                    n.async = false;
                    h.insertAdjacentElement('afterend', n);
                };
                h.insertAdjacentElement('afterend', t);
            })
        }
    })(window, document, 'script');
</script>
    
<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://sgtm.nature.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h === 'preview-www.nature.com') return;
            var e = d.createElement(t),
                s = d.getElementsByTagName(t)[0];
            e.setAttribute('crossorigin', 'anonymous');
            if (h === 'nature.com' || h.endsWith('.nature.com')) {
                e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-102.js';
                e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-8d962b73c2.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }
        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-b187e7f7c1.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-ac313c70d4.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-912fc24e80.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-b8f56acd2f.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-cfe28c12e7.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-41a1a3e87a.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning","description":"Plasmids are extrachromosomal DNA molecules that enable horizontal gene transfer in bacteria, often conferring advantages such as antibiotic resistance. Despite their importance, plasmids are underrepresented in genomic databases because of challenges in assembling them, caused by mosaicism and microdiversity. Current plasmid assemblers rely on detecting circular paths in single-sample assembly graphs but face limitations because of graph fragmentation, entanglement and low coverage. We introduce PlasMAAG (plasmid and organism metagenomic binning using assembly–alignment graphs), a method to recover plasmids and cellular genomes from metagenomic samples. PlasMAAG complements assembly graph signals across samples by generating an ‘assembly–alignment graph’, which is used alongside common binning features for improved plasmid reconstruction. On synthetic benchmark datasets, PlasMAAG reconstructed 50–121% more near-complete plasmids than competing methods and improved the Matthews correlation coefficient of geNomad contig classification by 28–106%. On hospital sewage samples, PlasMAAG outperformed competing methods, reconstructing 33% more plasmid sequences. PlasMAAG enables the study of organism–plasmid associations and intraplasmid diversity across samples. PlasMAAG uses cross-sample information to improve plasmid reconstruction from metagenomic samples.","datePublished":"2026-02-04T00:00:00Z","dateModified":"2026-02-04T00:00:00Z","pageStart":"1","pageEnd":"10","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41587-026-03005-7","keywords":["Classification and taxonomy","Machine learning","Life Sciences","general","Biotechnology","Biomedicine","Agriculture","Biomedical Engineering/Biotechnology","Bioinformatics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig5_HTML.png"],"isPartOf":{"name":"Nature Biotechnology","issn":["1546-1696","1087-0156"],"@type":["Periodical"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Pau Piera Líndez","url":"http://orcid.org/0000-0002-3058-5666","affiliation":[{"name":"University of Copenhagen","address":{"name":"Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lasse Schnell Danielsen","url":"http://orcid.org/0009-0006-2084-6363","affiliation":[{"name":"University of Copenhagen","address":{"name":"Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Iva Kovačić","url":"http://orcid.org/0009-0000-9974-5376","affiliation":[{"name":"University of Copenhagen","address":{"name":"Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marc Pielies Avellí","affiliation":[{"name":"University of Copenhagen","address":{"name":"Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Joseph Nesme","affiliation":[{"name":"University of Copenhagen","address":{"name":"Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lars Juhl Jensen","affiliation":[{"name":"University of Copenhagen","address":{"name":"Novo Nordisk Foundation Center for Protein Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jakob Nybo Andersen","url":"http://orcid.org/0000-0003-2860-7982","affiliation":[{"name":"University of Copenhagen","address":{"name":"Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"email":"jakob.nissen@sund.ku.dk","@type":"Person"},{"name":"Søren Johannes Sørensen","url":"http://orcid.org/0000-0001-6227-9906","affiliation":[{"name":"University of Copenhagen","address":{"name":"Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"email":"sjs@bio.ku.dk","@type":"Person"},{"name":"Simon Rasmussen","url":"http://orcid.org/0000-0001-6323-9041","affiliation":[{"name":"University of Copenhagen","address":{"name":"Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark","@type":"PostalAddress"},"@type":"Organization"}],"email":"srasmuss@sund.ku.dk","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41587-026-03005-7">
    
    
    <meta name="journal_id" content="41587"/>
    <meta name="dc.title" content="Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning"/>
    <meta name="dc.source" content="Nature Biotechnology 2026"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2026-02-04"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2026 The Author(s)"/>
    <meta name="dc.rights" content="2026 The Author(s)"/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="Plasmids are extrachromosomal DNA molecules that enable horizontal gene transfer in bacteria, often conferring advantages such as antibiotic resistance. Despite their importance, plasmids are underrepresented in genomic databases because of challenges in assembling them, caused by mosaicism and microdiversity. Current plasmid assemblers rely on detecting circular paths in single-sample assembly graphs but face limitations because of graph fragmentation, entanglement and low coverage. We introduce PlasMAAG (plasmid and organism metagenomic binning using assembly&#8211;alignment graphs), a method to recover plasmids and cellular genomes from metagenomic samples. PlasMAAG complements assembly graph signals across samples by generating an &#8216;assembly&#8211;alignment graph&#8217;, which is used alongside common binning features for improved plasmid reconstruction. On synthetic benchmark datasets, PlasMAAG reconstructed 50&#8211;121% more near-complete plasmids than competing methods and improved the Matthews correlation coefficient of geNomad contig classification by 28&#8211;106%. On hospital sewage samples, PlasMAAG outperformed competing methods, reconstructing 33% more plasmid sequences. PlasMAAG enables the study of organism&#8211;plasmid associations and intraplasmid diversity across samples. PlasMAAG uses cross-sample information to improve plasmid reconstruction from metagenomic samples."/>
    <meta name="prism.issn" content="1546-1696"/>
    <meta name="prism.publicationName" content="Nature Biotechnology"/>
    <meta name="prism.publicationDate" content="2026-02-04"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="1"/>
    <meta name="prism.endingPage" content="10"/>
    <meta name="prism.copyright" content="2026 The Author(s)"/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41587-026-03005-7"/>
    <meta name="prism.doi" content="doi:10.1038/s41587-026-03005-7"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41587-026-03005-7.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41587-026-03005-7"/>
    <meta name="citation_journal_title" content="Nature Biotechnology"/>
    <meta name="citation_journal_abbrev" content="Nat Biotechnol"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1696"/>
    <meta name="citation_title" content="Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning"/>
    <meta name="citation_online_date" content="2026/02/04"/>
    <meta name="citation_firstpage" content="1"/>
    <meta name="citation_lastpage" content="10"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_fulltext_world_readable" content=""/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41587-026-03005-7"/>
    <meta name="DOI" content="10.1038/s41587-026-03005-7"/>
    <meta name="size" content="206273"/>
    <meta name="citation_doi" content="10.1038/s41587-026-03005-7"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41587-026-03005-7&amp;api_key="/>
    <meta name="description" content="Plasmids are extrachromosomal DNA molecules that enable horizontal gene transfer in bacteria, often conferring advantages such as antibiotic resistance. Despite their importance, plasmids are underrepresented in genomic databases because of challenges in assembling them, caused by mosaicism and microdiversity. Current plasmid assemblers rely on detecting circular paths in single-sample assembly graphs but face limitations because of graph fragmentation, entanglement and low coverage. We introduce PlasMAAG (plasmid and organism metagenomic binning using assembly&#8211;alignment graphs), a method to recover plasmids and cellular genomes from metagenomic samples. PlasMAAG complements assembly graph signals across samples by generating an &#8216;assembly&#8211;alignment graph&#8217;, which is used alongside common binning features for improved plasmid reconstruction. On synthetic benchmark datasets, PlasMAAG reconstructed 50&#8211;121% more near-complete plasmids than competing methods and improved the Matthews correlation coefficient of geNomad contig classification by 28&#8211;106%. On hospital sewage samples, PlasMAAG outperformed competing methods, reconstructing 33% more plasmid sequences. PlasMAAG enables the study of organism&#8211;plasmid associations and intraplasmid diversity across samples. PlasMAAG uses cross-sample information to improve plasmid reconstruction from metagenomic samples."/>
    <meta name="dc.creator" content="Piera L&#237;ndez, Pau"/>
    <meta name="dc.creator" content="Danielsen, Lasse Schnell"/>
    <meta name="dc.creator" content="Kova&#269;i&#263;, Iva"/>
    <meta name="dc.creator" content="Pielies Avell&#237;, Marc"/>
    <meta name="dc.creator" content="Nesme, Joseph"/>
    <meta name="dc.creator" content="Jensen, Lars Juhl"/>
    <meta name="dc.creator" content="Andersen, Jakob Nybo"/>
    <meta name="dc.creator" content="S&#248;rensen, S&#248;ren Johannes"/>
    <meta name="dc.creator" content="Rasmussen, Simon"/>
    <meta name="dc.subject" content="Classification and taxonomy"/>
    <meta name="dc.subject" content="Machine learning"/>
    <meta name="citation_reference" content="citation_journal_title=Microb. Genom.; citation_title=Platon: identification and characterization of bacterial plasmid contigs in short-read draft assembliesexploiting protein sequence-based replicon distribution scores; citation_author=O Schwengers; citation_volume=6; citation_publication_date=2020; citation_pages=mgen000398; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.; citation_title=Beyond horizontal gene transfer: the role of plasmids in bacterial evolution; citation_author=J Rodr&#237;guez-Beltr&#225;n, J DelaFuente, R Le&#243;n-Sampedro, RC MacLean, &#193; San Mill&#225;n; citation_volume=19; citation_publication_date=2021; citation_pages=347-359; citation_doi=10.1038/s41579-020-00497-1; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Bacterial plasmids; citation_author=DJ Sherratt; citation_volume=3; citation_publication_date=1974; citation_pages=189-195; citation_doi=10.1016/0092-8674(74)90130-5; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Microbiol. Spectr.; citation_title=Fitness costs of plasmids: a limit to plasmid transmission; citation_author=A San Millan, RC MacLean; citation_volume=5; citation_publication_date=2017; citation_pages=5; citation_doi=10.1128/microbiolspec.MTBP-0016-2017; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.; citation_title=Mechanisms of, and barriers to, horizontal gene transfer between bacteria; citation_author=CM Thomas, KM Nielsen; citation_volume=3; citation_publication_date=2005; citation_pages=711-721; citation_doi=10.1038/nrmicro1234; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_title=Conjugative plasmids: vessels of the communal gene pool; citation_author=A Norman, LH Hansen, SJ S&#248;rensen; citation_volume=364; citation_publication_date=2009; citation_pages=2275-2289; citation_doi=10.1098/rstb.2009.0037; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Plasmid metagenome reveals high levels of antibiotic resistance genes and mobile genetic elements in activated sludge; citation_author=T Zhang, X-X Zhang, L Ye; citation_volume=6; citation_publication_date=2011; citation_doi=10.1371/journal.pone.0026041; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.; citation_title=Incorporating plasmid biology and metagenomics into a holistic model of the human gut microbiome; citation_author=SR Stockdale, C Hill; citation_volume=73; citation_publication_date=2023; citation_pages=102307; citation_doi=10.1016/j.mib.2023.102307; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Microbiology: metagenomics; citation_author=P Hugenholtz, GW Tyson; citation_volume=455; citation_publication_date=2008; citation_pages=481-483; citation_doi=10.1038/455481a; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Microbiol. Mol. Biol. Rev.; citation_title=Metagenomics: application of genomics to uncultured microorganisms; citation_author=J Handelsman; citation_volume=68; citation_publication_date=2004; citation_pages=669-685; citation_doi=10.1128/MMBR.68.4.669-685.2004; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Diverse plasmid systems and their ecology across human gut metagenomes revealed by PlasX and MobMess; citation_author=MK Yu, EC Fogarty, AM Eren; citation_volume=9; citation_publication_date=2024; citation_pages=830-847; citation_doi=10.1038/s41564-024-01610-3; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=mBio; citation_title=Plasmid-encoded traits vary across environments; citation_author=SS Finks, JBH Martiny; citation_volume=14; citation_publication_date=2023; citation_doi=10.1128/mbio.03191-22; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Plasmid; citation_title=Mosaic plasmids are abundant and unevenly distributed across prokaryotic taxa; citation_author=MW Pesesky, R Tilley, DAC Beck; citation_volume=102; citation_publication_date=2019; citation_pages=10-18; citation_doi=10.1016/j.plasmid.2019.02.003; citation_id=CR13"/>
    <meta name="citation_reference" content="Bouchot, J.-L. et al. in Computational Systems Biology (eds Kriete, A. &amp; Eils, R.) (Elsevier, 2014)."/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.; citation_title=High recombinant frequency in extraintestinal pathogenic Escherichia coli strains; citation_author=J Rodr&#237;guez-Beltr&#225;n; citation_volume=32; citation_publication_date=2015; citation_pages=1708-1716; citation_doi=10.1093/molbev/msv072; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.; citation_title=Towards a taxonomy of conjugative plasmids; citation_author=R Fernandez-Lopez, S Redondo, MP Garcillan-Barcia, F Cruz; citation_volume=38; citation_publication_date=2017; citation_pages=106-113; citation_doi=10.1016/j.mib.2017.05.005; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=SCAPP: an algorithm for improved plasmid assembly in metagenomes; citation_author=D Pellow; citation_volume=9; citation_publication_date=2021; citation_doi=10.1186/s40168-021-01068-z; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=Plasmid detection and assembly in genomic and metagenomic data sets; citation_author=D Antipov, M Raiko, A Lapidus, PA Pevzner; citation_volume=29; citation_publication_date=2019; citation_pages=961-968; citation_doi=10.1101/gr.241299.118; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Recycler: an algorithm for detecting plasmids from de novo assembly graphs; citation_author=R Rozov; citation_volume=33; citation_publication_date=2017; citation_pages=475-482; citation_doi=10.1093/bioinformatics/btw651; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Genomics Proteomics Bioinformatics; citation_title=Review of general algorithmic features for genome assemblers for next generation sequencers; citation_author=B Wajid, E Serpedin; citation_volume=10; citation_publication_date=2012; citation_pages=58-73; citation_doi=10.1016/j.gpb.2012.05.006; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.; citation_title=New approaches for metagenome assembly with short reads; citation_author=M Ayling, MD Clark, RM Leggett; citation_volume=21; citation_publication_date=2020; citation_pages=584-594; citation_doi=10.1093/bib/bbz020; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=Genomics; citation_title=Genomic mapping by fingerprinting random clones: a mathematical analysis; citation_author=ES Lander, MS Waterman; citation_volume=2; citation_publication_date=1988; citation_pages=231-239; citation_doi=10.1016/0888-7543(88)90007-9; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=STRONG: metagenomics strain resolution on assembly graphs; citation_author=C Quince; citation_volume=22; citation_publication_date=2021; citation_doi=10.1186/s13059-021-02419-7; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Improved metagenome binning and assembly using deep variational autoencoders; citation_author=JN Nissen; citation_volume=39; citation_publication_date=2021; citation_pages=555-560; citation_doi=10.1038/s41587-020-00777-4; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies; citation_author=DD Kang; citation_volume=7; citation_publication_date=2019; citation_doi=10.7717/peerj.7359; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets; citation_author=Y-W Wu, BA Simmons, SW Singer; citation_volume=32; citation_publication_date=2016; citation_pages=605-607; citation_doi=10.1093/bioinformatics/btv638; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SemiBin2: self-supervised contrastive learning leads to better MAGs for short- and long- read sequencing; citation_author=S Pan, X-M Zhao, LP Coelho; citation_volume=39; citation_publication_date=2023; citation_pages=i21-i29; citation_doi=10.1093/bioinformatics/btad209; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=A deep siamese neural network improves metagenome-assembled genomes in microbiome datasets across different environments; citation_author=S Pan, C Zhu, X-M Zhao, LP Coelho; citation_volume=13; citation_publication_date=2022; citation_doi=10.1038/s41467-022-29843-y; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Commun. Biol.; citation_title=Adversarial and variational autoencoders improve metagenomic binning; citation_author=PP L&#237;ndez; citation_volume=6; citation_publication_date=2023; citation_pages=1073; citation_doi=10.1038/s42003-023-05452-3; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Metagenomic binning with assembly graph embeddings; citation_author=A Lamurias, M Sereika, M Albertsen, K Hose, TD Nielsen; citation_volume=38; citation_publication_date=2022; citation_pages=4481-4487; citation_doi=10.1093/bioinformatics/btac557; citation_id=CR30"/>
    <meta name="citation_reference" content="Kutuzova, S. et al. Binning meets taxonomy: TaxVAMB improves metagenome binning using bi-modal variational autoencoder. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.10.25.620172
                
               (2024)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Genome binning of viral entities from bulk metagenomics data; citation_author=J Johansen; citation_volume=13; citation_publication_date=2022; citation_pages=965; citation_doi=10.1038/s41467-022-28581-5; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Taxometer: improving taxonomic classification of metagenomics contigs; citation_author=S Kutuzova, M Nielsen, P Piera, JN Nissen, S Rasmussen; citation_volume=15; citation_publication_date=2024; citation_doi=10.1038/s41467-024-52771-y; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=A comparison of single-coverage and multi-coverage metagenomic binning reveals extensive hidden contamination; citation_author=J Mattock, M Watson; citation_volume=20; citation_publication_date=2023; citation_pages=1170-1173; citation_doi=10.1038/s41592-023-01934-8; citation_id=CR34"/>
    <meta name="citation_reference" content="Abraham, L. louisabraham/fastnode2vec version-0.0.5 Zenodo 
                https://doi.org/10.5281/zenodo.3902632
                
               (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Identification of mobile genetic elements with geNomad; citation_author=AP Camargo; citation_volume=42; citation_publication_date=2024; citation_pages=1303-1312; citation_doi=10.1038/s41587-023-01953-y; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Critical Assessment of Metagenome Interpretation&#8212;a benchmark of metagenomics software; citation_author=A Sczyrba; citation_volume=14; citation_publication_date=2017; citation_pages=1063-1071; citation_doi=10.1038/nmeth.4458; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=GraphBin: refined binning of metagenomic contigs using assembly graphs; citation_author=V Mallawaarachchi, A Wickramarachchi, Y Lin; citation_volume=36; citation_publication_date=2020; citation_pages=3307-3313; citation_doi=10.1093/bioinformatics/btaa180; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.; citation_title=Solving genomic puzzles: computational methods for metagenomic binning; citation_author=V Mallawaarachchi; citation_volume=25; citation_publication_date=2024; citation_pages=bbae372 ; citation_doi=10.1093/bib/bbae372; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Environ. Int.; citation_title=Insights into the circular: the cryptic plasmidome and its derived antibiotic resistome in the urban water systems; citation_author=Z Yu; citation_volume=183; citation_publication_date=2024; citation_doi=10.1016/j.envint.2023.108351; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=The mosaic architecture of Aeromonas salmonicida subsp. salmonicida pAsa4 plasmid and its consequences on antibiotic resistance; citation_author=KH Tanaka; citation_volume=4; citation_publication_date=2016; citation_doi=10.7717/peerj.2595; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=metaSPAdes: a new versatile metagenomic assembler; citation_author=S Nurk, D Meleshko, A Korobeynikov, PA Pevzner; citation_volume=27; citation_publication_date=2017; citation_pages=824-834; citation_doi=10.1101/gr.213959.116; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Minimap2: pairwise alignment for nucleotide sequences; citation_author=H Li; citation_volume=34; citation_publication_date=2018; citation_pages=3094-3100; citation_doi=10.1093/bioinformatics/bty191; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=H Li; citation_volume=25; citation_publication_date=2009; citation_pages=2078-2079; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.; citation_title=Basic local alignment search tool; citation_author=SF Altschul, W Gish, W Miller, EW Myers, DJ Lipman; citation_volume=215; citation_publication_date=1990; citation_pages=403-410; citation_doi=10.1016/S0022-2836(05)80360-2; citation_id=CR45"/>
    <meta name="citation_reference" content="Grover, A. &amp; Leskovec, J. Node2vec: scalable feature learning for networks. Preprint at 
                https://doi.org/10.48550/arXiv.1607.00653
                
               (2016)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome; citation_author=S Nayfach; citation_volume=6; citation_publication_date=2021; citation_pages=960-970; citation_doi=10.1038/s41564-021-00928-6; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Critical Assessment of Metagenome Interpretation: the second round of challenges; citation_author=F Meyer; citation_volume=19; citation_publication_date=2022; citation_pages=429-440; citation_doi=10.1038/s41592-022-01431-4; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=CAMISIM: simulating metagenomes and microbial communities; citation_author=A Fritz; citation_volume=7; citation_publication_date=2019; citation_doi=10.1186/s40168-019-0633-6; citation_id=CR49"/>
    <meta name="citation_reference" content="Hagberg, D. &amp; Pieter, J. Exploring network structure, dynamics, function using NetworkX. In Proc. 24th Science in Python Conference (eds Ambade, A. et al.) (SciPy, 2025)."/>
    <meta name="citation_reference" content="Nissen, J. N., Lind&#233;z, P. P. &amp; Rasmussen, S. BinBencher: fast, flexible and meaningful benchmarking suite for metagenomic binning. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.06.592671
                
               (2024)."/>
    <meta name="citation_reference" content="citation_journal_title=Plasmid; citation_title=An improved direct metamobilome approach increases the detection of larger-sized circular elements across kingdoms; citation_author=KWS Alanin; citation_volume=115; citation_publication_date=2021; citation_pages=102576; citation_doi=10.1016/j.plasmid.2021.102576; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Insights into the bovine rumen plasmidome; citation_author=AB Kav; citation_volume=109; citation_publication_date=2012; citation_pages=5452-5457; citation_doi=10.1073/pnas.1116410109; citation_id=CR53"/>
    <meta name="citation_reference" content="Green, M. R. &amp; Sambrook, J. Molecular Cloning: A Laboratory Manual 4th edn (Cold Spring Harbor Laboratory, 2012)."/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=GTDB-Tk v2: memory friendly classification with the genome taxonomy database; citation_author=P-A Chaumeil, AJ Mussig, P Hugenholtz, DH Parks; citation_volume=38; citation_publication_date=2022; citation_pages=5315-5316; citation_doi=10.1093/bioinformatics/btac672; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=The PLSDB 2025 update: enhanced annotations and improved functionality for comprehensive plasmid research; citation_author=L-AG Molano, P Hirsch, M Hannig, R M&#252;ller, A Keller; citation_volume=53; citation_publication_date=2025; citation_pages=D189-D196; citation_doi=10.1093/nar/gkae1095; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Community-led, integrated, reproducible multi-omics with anvi&#8217;o; citation_author=AM Eren; citation_volume=6; citation_publication_date=2021; citation_pages=3-6; citation_doi=10.1038/s41564-020-00834-3; citation_id=CR57"/>
    <meta name="citation_author" content="Piera L&#237;ndez, Pau"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Danielsen, Lasse Schnell"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Kova&#269;i&#263;, Iva"/>
    <meta name="citation_author_institution" content="Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Pielies Avell&#237;, Marc"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Nesme, Joseph"/>
    <meta name="citation_author_institution" content="Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Jensen, Lars Juhl"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Protein Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Andersen, Jakob Nybo"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="S&#248;rensen, S&#248;ren Johannes"/>
    <meta name="citation_author_institution" content="Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="citation_author" content="Rasmussen, Simon"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@NatureBiotech"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning"/>
    <meta name="twitter:description" content="Nature Biotechnology - PlasMAAG uses cross-sample information to improve plasmid reconstruction from metagenomic samples."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41587-026-03005-7"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning - Nature Biotechnology"/>
    <meta property="og:description" content="PlasMAAG uses cross-sample information to improve plasmid reconstruction from metagenomic samples."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
</head>
<body class="article-page">

<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/biotech.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41587-026-03005-7;doi=10.1038/s41587-026-03005-7;subjmeta=114,1305,1386,631;kwrd=Classification+and+taxonomy,Machine+learning">
        
        <script>
            window.SN = window.SN || {};
            window.SN.libs = window.SN.libs || {};
            window.SN.libs.ads = window.SN.libs.ads || {};
            window.SN.libs.ads.slotConfig = window.SN.libs.ads.slotConfig || {};
            
                window.SN.libs.ads.slotConfig['top'] = {
                    'pos': 'top',
                    'type': 'article',
                    'path': 's41587-026-03005-7'
                };
            
            
            window.SN.libs.ads.slotConfig['kwrd'] = 'Classification+and+taxonomy,Machine+learning';
            
            
            window.SN.libs.ads.slotConfig['subjmeta'] = '114,1305,1386,631';
            
            
        </script>
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/biotech.nature.com/article&amp;sz=728x90&amp;c=-1115682351&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41587-026-03005-7%26doi%3D10.1038/s41587-026-03005-7%26subjmeta%3D114,1305,1386,631%26kwrd%3DClassification+and+taxonomy,Machine+learning">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/biotech.nature.com/article&amp;sz=728x90&amp;c=-1115682351&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41587-026-03005-7%26doi%3D10.1038/s41587-026-03005-7%26subjmeta%3D114,1305,1386,631%26kwrd%3DClassification+and+taxonomy,Machine+learning"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#efd600">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nbt"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nbt/header-a6bd3dd1eeb5a9d6dcc36be30996a59f.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nbt/header-8f661e82c86879a1b8f03cbd053d5155.svg" height="32" alt="Nature Biotechnology">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41587-026-03005-7?error=cookies_not_supported&code=4e0b56f5-e67b-4946-9082-129f4940984c'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span class="c-header__show-text-sm">Content</span>
                                        <span class="c-header__show-text">Explore content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link"
                                       href="https://journal-alerts.springernature.com/subscribe?journal_id&#x3D;41587"
                                       rel="nofollow"
                                       data-track="nav_sign_up_for_alerts"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/nbt.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/nbt" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature biotechnology"><span itemprop="name">nature biotechnology</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/nbt/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide"
                     id="js-enable-context-bar"
                     data-test="context-bar"
                     data-context-bar
                     aria-hidden="true">
                    <div class="c-context-bar__container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning
                        </div>
                        <div class="c-context-bar__cta-container">
                            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41587-026-03005-7.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                            
                        </div>
                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-8 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                <div class="app-cta-group">
                                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41587-026-03005-7.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                                    
                                </div>
                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-science/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2026-02-04">04 February 2026</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="1_9" data-track-context="researcher popup with no profile" href="#auth-Pau-Piera_L_ndez-Aff1" data-author-popup="auth-Pau-Piera_L_ndez-Aff1" data-author-search="Piera Líndez, Pau">Pau Piera Líndez</a><span class="u-js-hide"> 
            <a class="js-orcid" href="https://orcid.org/0000-0002-3058-5666"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3058-5666</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="2_9" data-track-context="researcher popup with no profile" href="#auth-Lasse_Schnell-Danielsen-Aff1" data-author-popup="auth-Lasse_Schnell-Danielsen-Aff1" data-author-search="Danielsen, Lasse Schnell">Lasse Schnell Danielsen</a><span class="u-js-hide"> 
            <a class="js-orcid" href="https://orcid.org/0009-0006-2084-6363"><span class="u-visually-hidden">ORCID: </span>orcid.org/0009-0006-2084-6363</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="3_9" data-track-context="researcher popup with no profile" href="#auth-Iva-Kova_i_-Aff2" data-author-popup="auth-Iva-Kova_i_-Aff2" data-author-search="Kovačić, Iva">Iva Kovačić</a><span class="u-js-hide"> 
            <a class="js-orcid" href="https://orcid.org/0009-0000-9974-5376"><span class="u-visually-hidden">ORCID: </span>orcid.org/0009-0000-9974-5376</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="4_9" data-track-context="researcher popup with no profile" href="#auth-Marc-Pielies_Avell_-Aff1" data-author-popup="auth-Marc-Pielies_Avell_-Aff1" data-author-search="Pielies Avellí, Marc">Marc Pielies Avellí</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="5_9" data-track-context="researcher popup with no profile" href="#auth-Joseph-Nesme-Aff2" data-author-popup="auth-Joseph-Nesme-Aff2" data-author-search="Nesme, Joseph">Joseph Nesme</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="6_9" data-track-context="researcher popup with no profile" href="#auth-Lars_Juhl-Jensen-Aff3" data-author-popup="auth-Lars_Juhl-Jensen-Aff3" data-author-search="Jensen, Lars Juhl">Lars Juhl Jensen</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="7_9" data-track-context="researcher popup with no profile" href="#auth-Jakob_Nybo-Andersen-Aff1" data-author-popup="auth-Jakob_Nybo-Andersen-Aff1" data-author-search="Andersen, Jakob Nybo" data-corresp-id="c1">Jakob Nybo Andersen<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="https://orcid.org/0000-0003-2860-7982"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2860-7982</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="8_9" data-track-context="researcher popup with no profile" href="#auth-S_ren_Johannes-S_rensen-Aff2" data-author-popup="auth-S_ren_Johannes-S_rensen-Aff2" data-author-search="Sørensen, Søren Johannes" data-corresp-id="c2">Søren Johannes Sørensen<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="https://orcid.org/0000-0001-6227-9906"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6227-9906</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 9 authors for this article" title="Show all 9 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" data-track-index="9_9" data-track-context="researcher popup with no profile" href="#auth-Simon-Rasmussen-Aff1" data-author-popup="auth-Simon-Rasmussen-Aff1" data-author-search="Rasmussen, Simon" data-corresp-id="c3">Simon Rasmussen<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="https://orcid.org/0000-0001-6323-9041"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6323-9041</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nbt" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Biotechnology</i></a>

                         (<span data-test="article-publication-year">2026</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/classification-and-taxonomy" data-track="click" data-track-action="view subject" data-track-label="link">Classification and taxonomy</a></li><li class="c-article-subject-list__subject"><a href="/subjects/machine-learning" data-track="click" data-track-action="view subject" data-track-label="link">Machine learning</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Plasmids are extrachromosomal DNA molecules that enable horizontal gene transfer in bacteria, often conferring advantages such as antibiotic resistance. Despite their importance, plasmids are underrepresented in genomic databases because of challenges in assembling them, caused by mosaicism and microdiversity. Current plasmid assemblers rely on detecting circular paths in single-sample assembly graphs but face limitations because of graph fragmentation, entanglement and low coverage. We introduce PlasMAAG (plasmid and organism metagenomic binning using assembly–alignment graphs), a method to recover plasmids and cellular genomes from metagenomic samples. PlasMAAG complements assembly graph signals across samples by generating an ‘assembly–alignment graph’, which is used alongside common binning features for improved plasmid reconstruction. On synthetic benchmark datasets, PlasMAAG reconstructed 50–121% more near-complete plasmids than competing methods and improved the Matthews correlation coefficient of geNomad contig classification by 28–106%. On hospital sewage samples, PlasMAAG outperformed competing methods, reconstructing 33% more plasmid sequences. PlasMAAG enables the study of organism–plasmid associations and intraplasmid diversity across samples.</p></div></div></section>

            
                
            

            
                
                    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Plasmids are extrachromosomal DNA molecules within a host cell that are physically separated from chromosomal DNA and can replicate independently<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schwengers, O. et al. Platon: identification and characterization of bacterial plasmid contigs in short-read draft assembliesexploiting protein sequence-based replicon distribution scores. Microb. Genom. 6, mgen000398 (2020)." href="#ref-CR1" id="ref-link-section-d164666590e487">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rodríguez-Beltrán, J., DelaFuente, J., León-Sampedro, R., MacLean, R. C. &amp; San Millán, Á Beyond horizontal gene transfer: the role of plasmids in bacterial evolution. Nat. Rev. Microbiol. 19, 347–359 (2021)." href="#ref-CR2" id="ref-link-section-d164666590e487_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Sherratt, D. J. Bacterial plasmids. Cell 3, 189–195 (1974)." href="/articles/s41587-026-03005-7#ref-CR3" id="ref-link-section-d164666590e490">3</a></sup>. Plasmids differ in genome length, copy number, replication mechanism and conjugation mode. The part of the plasmid that encodes the core replication machinery is typically contiguous and is called the ‘backbone’. The replication and maintenance of plasmids incur a metabolic burden for the host and, to avoid purifying selection, the plasmid must carry additional ‘cargo’ genes that increase the fitness of either the host or the plasmid<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="San Millan, A. &amp; MacLean, R. C. Fitness costs of plasmids: a limit to plasmid transmission. Microbiol. Spectr. 5, 5 (2017)." href="/articles/s41587-026-03005-7#ref-CR4" id="ref-link-section-d164666590e494">4</a></sup>. These may be genes that confer antibiotic resistance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Thomas, C. M. &amp; Nielsen, K. M. Mechanisms of, and barriers to, horizontal gene transfer between bacteria. Nat. Rev. Microbiol. 3, 711–721 (2005)." href="#ref-CR5" id="ref-link-section-d164666590e498">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Norman, A., Hansen, L. H. &amp; Sørensen, S. J. Conjugative plasmids: vessels of the communal gene pool. Philos. Trans. R. Soc. Lond. B Biol. Sci. 364, 2275–2289 (2009)." href="#ref-CR6" id="ref-link-section-d164666590e498_1">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Zhang, T., Zhang, X.-X. &amp; Ye, L. Plasmid metagenome reveals high levels of antibiotic resistance genes and mobile genetic elements in activated sludge. PLoS ONE 6, e26041 (2011)." href="/articles/s41587-026-03005-7#ref-CR7" id="ref-link-section-d164666590e501">7</a></sup>.</p><p>Approximately 50% of bacteria carry one or more plasmids<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Stockdale, S. R. &amp; Hill, C. Incorporating plasmid biology and metagenomics into a holistic model of the human gut microbiome. Curr. Opin. Microbiol. 73, 102307 (2023)." href="/articles/s41587-026-03005-7#ref-CR8" id="ref-link-section-d164666590e508">8</a></sup>. Nonetheless, in databases, sequences from plasmids remain underrepresented compared to those from cellular genomes. For instance, RefSeq (<a href="https://www.ncbi.nlm.nih.gov/refseq/">https://www.ncbi.nlm.nih.gov/refseq/</a>) contains 82,471 bacterial genomes but only 7,892 plasmids. Characterization of environmental plasmids in in vitro conditions is inherently limited by the ‘cultivation bottleneck’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Hugenholtz, P. &amp; Tyson, G. W. Microbiology: metagenomics. Nature 455, 481–483 (2008)." href="/articles/s41587-026-03005-7#ref-CR9" id="ref-link-section-d164666590e519">9</a></sup><sup>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Handelsman, J. Metagenomics: application of genomics to uncultured microorganisms. Microbiol. Mol. Biol. Rev. 68, 669–685 (2004)." href="/articles/s41587-026-03005-7#ref-CR10" id="ref-link-section-d164666590e523">10</a></sup>, where laboratory conditions modify microbial diversity, offering a poor representation of the original composition. Therefore, despite the great number of plasmid-related studies, most studies have investigated plasmid virulence and properties in isolated strains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Stockdale, S. R. &amp; Hill, C. Incorporating plasmid biology and metagenomics into a holistic model of the human gut microbiome. Curr. Opin. Microbiol. 73, 102307 (2023)." href="/articles/s41587-026-03005-7#ref-CR8" id="ref-link-section-d164666590e527">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yu, M. K., Fogarty, E. C. &amp; Eren, A. M. Diverse plasmid systems and their ecology across human gut metagenomes revealed by PlasX and MobMess. Nat. Microbiol. 9, 830–847 (2024)." href="#ref-CR11" id="ref-link-section-d164666590e530">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Finks, S. S. &amp; Martiny, J. B. H. Plasmid-encoded traits vary across environments. mBio 14, e0319122 (2023)." href="#ref-CR12" id="ref-link-section-d164666590e530_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Pesesky, M. W., Tilley, R. &amp; Beck, D. A. C. Mosaic plasmids are abundant and unevenly distributed across prokaryotic taxa. Plasmid 102, 10–18 (2019)." href="/articles/s41587-026-03005-7#ref-CR13" id="ref-link-section-d164666590e533">13</a></sup>. This mismatch between estimated plasmid prevalence in bacteria and plasmid representation in the databases emphasizes our incomplete understanding of the plasmid genetic structure, diversity, and function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Rodríguez-Beltrán, J., DelaFuente, J., León-Sampedro, R., MacLean, R. C. &amp; San Millán, Á Beyond horizontal gene transfer: the role of plasmids in bacterial evolution. Nat. Rev. Microbiol. 19, 347–359 (2021)." href="/articles/s41587-026-03005-7#ref-CR2" id="ref-link-section-d164666590e537">2</a></sup>.</p><p>Metagenomics offers culture free alternative techniques; however, the genetic complexity of environmental samples complicates the process<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Hugenholtz, P. &amp; Tyson, G. W. Microbiology: metagenomics. Nature 455, 481–483 (2008)." href="/articles/s41587-026-03005-7#ref-CR9" id="ref-link-section-d164666590e544">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Handelsman, J. Metagenomics: application of genomics to uncultured microorganisms. Microbiol. Mol. Biol. Rev. 68, 669–685 (2004)." href="/articles/s41587-026-03005-7#ref-CR10" id="ref-link-section-d164666590e547">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Bouchot, J.-L. et al. in Computational Systems Biology (eds Kriete, A. &amp; Eils, R.) (Elsevier, 2014)." href="/articles/s41587-026-03005-7#ref-CR14" id="ref-link-section-d164666590e550">14</a></sup>. In addition to the challenges of assembling bacterial chromosomes, assembly of plasmids brings other challenges. First, plasmids undergo frequent recombination, creating groups of plasmids that share a backbone but diverge on their cargo sequence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yu, M. K., Fogarty, E. C. &amp; Eren, A. M. Diverse plasmid systems and their ecology across human gut metagenomes revealed by PlasX and MobMess. Nat. Microbiol. 9, 830–847 (2024)." href="/articles/s41587-026-03005-7#ref-CR11" id="ref-link-section-d164666590e554">11</a></sup>. Second, plasmids at high copy number have higher mutation rates than chromosomes, which increases microdiversity and makes them difficult to assemble with de Bruijn graph-based assemblers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Rodríguez-Beltrán, J. et al. High recombinant frequency in extraintestinal pathogenic Escherichia coli strains. Mol. Biol. Evol. 32, 1708–1716 (2015)." href="/articles/s41587-026-03005-7#ref-CR15" id="ref-link-section-d164666590e558">15</a></sup>. Third, plasmids are enriched for repeated sequences associated with transposable elements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Rodríguez-Beltrán, J., DelaFuente, J., León-Sampedro, R., MacLean, R. C. &amp; San Millán, Á Beyond horizontal gene transfer: the role of plasmids in bacterial evolution. Nat. Rev. Microbiol. 19, 347–359 (2021)." href="/articles/s41587-026-03005-7#ref-CR2" id="ref-link-section-d164666590e562">2</a></sup>. A consequence of this is that plasmid sequences will be fragmented across assemblies and entangled by sharing the same backbone and repeated genetic elements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yu, M. K., Fogarty, E. C. &amp; Eren, A. M. Diverse plasmid systems and their ecology across human gut metagenomes revealed by PlasX and MobMess. Nat. Microbiol. 9, 830–847 (2024)." href="/articles/s41587-026-03005-7#ref-CR11" id="ref-link-section-d164666590e566">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Fernandez-Lopez, R., Redondo, S., Garcillan-Barcia, M. P. &amp; de la Cruz, F. Towards a taxonomy of conjugative plasmids. Curr. Opin. Microbiol. 38, 106–113 (2017)." href="/articles/s41587-026-03005-7#ref-CR16" id="ref-link-section-d164666590e569">16</a></sup>.</p><p>To overcome these challenges, dedicated metagenomic plasmid assemblers such as Recycler, metaplasmidSPAdes and SCAPP have been developed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pellow, D. et al. SCAPP: an algorithm for improved plasmid assembly in metagenomes. Microbiome 9, 144 (2021)." href="#ref-CR17" id="ref-link-section-d164666590e576">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Antipov, D., Raiko, M., Lapidus, A. &amp; Pevzner, P. A. Plasmid detection and assembly in genomic and metagenomic data sets. Genome Res. 29, 961–968 (2019)." href="#ref-CR18" id="ref-link-section-d164666590e576_1">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Rozov, R. et al. Recycler: an algorithm for detecting plasmids from de novo assembly graphs. Bioinformatics 33, 475–482 (2017)." href="/articles/s41587-026-03005-7#ref-CR19" id="ref-link-section-d164666590e579">19</a></sup>. These methods rely on the assembly graph, a data structure used by metagenome assemblers, that represents overlaps between sequencing reads. Assembly graphs represent contiguous sequences (contigs) as nodes and overlaps between these as edges<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Wajid, B. &amp; Serpedin, E. Review of general algorithmic features for genome assemblers for next generation sequencers. Genomics Proteomics Bioinformatics 10, 58–73 (2012)." href="/articles/s41587-026-03005-7#ref-CR20" id="ref-link-section-d164666590e583">20</a></sup>. By leveraging assembly graphs, plasmid assemblers can identify connected sequences and resolve complex genomic regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Ayling, M., Clark, M. D. &amp; Leggett, R. M. New approaches for metagenome assembly with short reads. Brief. Bioinform. 21, 584–594 (2020)." href="/articles/s41587-026-03005-7#ref-CR21" id="ref-link-section-d164666590e587">21</a></sup>. Common to existing methods is that they operate on single-sample assembly graphs, using the circularity of plasmids and contig coverage. However, the methods have fundamental limitations. First, low coverage causes the ‘fragmentation problem’, where some plasmids appear disconnected in the graph, making them impossible to identify by graph peeling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Pesesky, M. W., Tilley, R. &amp; Beck, D. A. C. Mosaic plasmids are abundant and unevenly distributed across prokaryotic taxa. Plasmid 102, 10–18 (2019)." href="/articles/s41587-026-03005-7#ref-CR13" id="ref-link-section-d164666590e591">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Lander, E. S. &amp; Waterman, M. S. Genomic mapping by fingerprinting random clones: a mathematical analysis. Genomics 2, 231–239 (1988)." href="/articles/s41587-026-03005-7#ref-CR22" id="ref-link-section-d164666590e594">22</a></sup>. Second, the high recombination rate of plasmids causes entangled assembly graph structures that thwart the graph cycle detection approaches of MetaplasmidSPAdes and SCAPP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Quince, C. et al. STRONG: metagenomics strain resolution on assembly graphs. Genome Biol. 22, 214 (2021)." href="/articles/s41587-026-03005-7#ref-CR23" id="ref-link-section-d164666590e598">23</a></sup>. Lastly, SCAPP and metaplasmidSPAdes leverage plasmid gene signatures to guide the plasmid candidate’s path extraction from the assembly graph, making them less sensitive to uncharacterized plasmids.</p><p>Binning is a computational strategy used to reconstruct genomes by grouping contigs on the basis of their genome of origin, providing an alternative to assembly graph-based methods. Modern binners typically integrate several sequence features, including <i>k</i>-mer composition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nissen, J. N. et al. Improved metagenome binning and assembly using deep variational autoencoders. Nat. Biotechnol. 39, 555–560 (2021)." href="#ref-CR24" id="ref-link-section-d164666590e609">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. PeerJ 7, e7359 (2019)." href="#ref-CR25" id="ref-link-section-d164666590e609_1">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wu, Y.-W., Simmons, B. A. &amp; Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. Bioinformatics 32, 605–607 (2016)." href="#ref-CR26" id="ref-link-section-d164666590e609_2">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pan, S., Zhao, X.-M. &amp; Coelho, L. P. SemiBin2: self-supervised contrastive learning leads to better MAGs for short- and long- read sequencing. Bioinformatics 39, i21–i29 (2023)." href="#ref-CR27" id="ref-link-section-d164666590e609_3">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pan, S., Zhu, C., Zhao, X.-M. &amp; Coelho, L. P. A deep siamese neural network improves metagenome-assembled genomes in microbiome datasets across different environments. Nat. Commun. 13, 2326 (2022)." href="#ref-CR28" id="ref-link-section-d164666590e609_4">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Líndez, P. P. et al. Adversarial and variational autoencoders improve metagenomic binning. Commun. Biol. 6, 1073 (2023)." href="/articles/s41587-026-03005-7#ref-CR29" id="ref-link-section-d164666590e612">29</a></sup>, abundance patterns across samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nissen, J. N. et al. Improved metagenome binning and assembly using deep variational autoencoders. Nat. Biotechnol. 39, 555–560 (2021)." href="#ref-CR24" id="ref-link-section-d164666590e616">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. PeerJ 7, e7359 (2019)." href="#ref-CR25" id="ref-link-section-d164666590e616_1">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wu, Y.-W., Simmons, B. A. &amp; Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. Bioinformatics 32, 605–607 (2016)." href="#ref-CR26" id="ref-link-section-d164666590e616_2">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pan, S., Zhao, X.-M. &amp; Coelho, L. P. SemiBin2: self-supervised contrastive learning leads to better MAGs for short- and long- read sequencing. Bioinformatics 39, i21–i29 (2023)." href="#ref-CR27" id="ref-link-section-d164666590e616_3">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pan, S., Zhu, C., Zhao, X.-M. &amp; Coelho, L. P. A deep siamese neural network improves metagenome-assembled genomes in microbiome datasets across different environments. Nat. Commun. 13, 2326 (2022)." href="#ref-CR28" id="ref-link-section-d164666590e616_4">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Líndez, P. P. et al. Adversarial and variational autoencoders improve metagenomic binning. Commun. Biol. 6, 1073 (2023)." href="/articles/s41587-026-03005-7#ref-CR29" id="ref-link-section-d164666590e619">29</a></sup>, assembly graph connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Lamurias, A., Sereika, M., Albertsen, M., Hose, K. &amp; Nielsen, T. D. Metagenomic binning with assembly graph embeddings. Bioinformatics 38, 4481–4487 (2022)." href="/articles/s41587-026-03005-7#ref-CR30" id="ref-link-section-d164666590e623">30</a></sup> and taxonomic markers or annotations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Wu, Y.-W., Simmons, B. A. &amp; Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. Bioinformatics 32, 605–607 (2016)." href="/articles/s41587-026-03005-7#ref-CR26" id="ref-link-section-d164666590e627">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Pan, S., Zhu, C., Zhao, X.-M. &amp; Coelho, L. P. A deep siamese neural network improves metagenome-assembled genomes in microbiome datasets across different environments. Nat. Commun. 13, 2326 (2022)." href="/articles/s41587-026-03005-7#ref-CR28" id="ref-link-section-d164666590e630">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Kutuzova, S. et al. Binning meets taxonomy: TaxVAMB improves metagenome binning using bi-modal variational autoencoder. Preprint at bioRxiv &#xA;                https://doi.org/10.1101/2024.10.25.620172&#xA;                &#xA;               (2024)." href="/articles/s41587-026-03005-7#ref-CR31" id="ref-link-section-d164666590e633">31</a></sup>. Most of these features can be computed on a per-sequence basis and are, therefore, not vulnerable to the fragmentation problem suffered by assembly graphs with low coverage. Furthermore, it has been shown how binning information can be used to refine contig classification, using binning features to guide classification rather than contig classification to reconstruct the original sequences<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Johansen, J. et al. Genome binning of viral entities from bulk metagenomics data. Nat. Commun. 13, 965 (2022)." href="/articles/s41587-026-03005-7#ref-CR32" id="ref-link-section-d164666590e638">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Kutuzova, S., Nielsen, M., Piera, P., Nissen, J. N. &amp; Rasmussen, S. Taxometer: improving taxonomic classification of metagenomics contigs. Nat. Commun. 15, 8357 (2024)." href="/articles/s41587-026-03005-7#ref-CR33" id="ref-link-section-d164666590e641">33</a></sup>. We previously developed the binning tool VAMB (variational autoencoder (VAE) for metagenomic binning), which combines several of these features using a VAE into a latent space that is clustered to form bins<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Nissen, J. N. et al. Improved metagenome binning and assembly using deep variational autoencoders. Nat. Biotechnol. 39, 555–560 (2021)." href="/articles/s41587-026-03005-7#ref-CR24" id="ref-link-section-d164666590e645">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Mattock, J. &amp; Watson, M. A comparison of single-coverage and multi-coverage metagenomic binning reveals extensive hidden contamination. Nat. Methods 20, 1170–1173 (2023)." href="/articles/s41587-026-03005-7#ref-CR34" id="ref-link-section-d164666590e648">34</a></sup>.</p><p>Here, we introduce assembly–alignment graphs (AAGs), which combine intrasample sequence overlaps recorded by assembly graphs with cross-sample overlaps detected by ordinary sequence alignment. Using a contrastive learning approach, we integrated AAGs with standard binning features in a new binning framework called PlasMAAG that can reconstruct both plasmids and cellular genomes. Across simulated and real datasets, PlasMAAG consistently outperformed existing binning approaches, particularly improving plasmid reconstruction while maintaining competitive performance on cellular genomes. In hospital sewage samples, PlasMAAG recovered substantially more plasmid sequences than other tools, validated using paired long-read and short-read data. By jointly reconstructing plasmids and their hosts, PlasMAAG enables downstream analyses of host–plasmid associations and plasmid diversity across samples. PlasMAAG supports ‘multisample’ characterization of both plasmids and cellular genomes, achieving state-of-the-art plasmid reconstruction with reduced computational requirements.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Overview of PlasMAAG</h3><p>The PlasMAAG pipeline is designed to recover plasmids and cellular genomes from metagenomics samples (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig1">1</a>). Compared to our previous developed binning algorithm VAMB, PlasMAAG introduces three novelties. First, we use contig alignment graphs to combine per-sample assembly graphs from several samples into a single graph called an AAG. The AAG is then projected to an embedding space with fastnode2vec<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Abraham, L. louisabraham/fastnode2vec version-0.0.5 Zenodo &#xA;                https://doi.org/10.5281/zenodo.3902632&#xA;                &#xA;               (2020)." href="/articles/s41587-026-03005-7#ref-CR35" id="ref-link-section-d164666590e670">35</a></sup>, from which communities of contigs can be extracted. Second, we enhanced the training of the VAE by adding contrastive learning on the basis of information from the AAG. Third, we use binning to ensemble geNomad<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Camargo, A. P. et al. Identification of mobile genetic elements with geNomad. Nat. Biotechnol. 42, 1303–1312 (2024)." href="/articles/s41587-026-03005-7#ref-CR36" id="ref-link-section-d164666590e674">36</a></sup> contig annotation scores across bins to classify the bins into plasmid or cellular genomes.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="PlasMAAG workflow overview."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: PlasMAAG workflow overview.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="687"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>PlasMAAG leverages assembly graphs, alignment graphs, <i>k</i>-mer signal and contig coabundances for binning, with a final step where bins are classified as cellular or plasmid bins on the basis of refined geNomad predictions. <b>a</b>, Per-sample assembly graphs are merged with the between-sample alignment graph, generating the AAG. <b>b</b>, Fastnode2vec is used to generate contig embeddings from the AAG, from where contig communities are extracted. Communities are expanded, merged and purified using a VAE with contrastive loss that pushes communities toward being preserved in the embedding. TNF, tetranucleotide frequency. <b>c</b>, Plasmid and cellular candidate bins are extracted from the VAE embedding on the basis of their geNomad scores, using distinct plasmid and cellular clustering strategies.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">PlasMAAG reconstructed 21–212% more plasmid bins compared to SCAPP confident</h3><p>To develop and test PlasMAAG, we reassembled the simulated Critical Assessment of Metagenome Interpretation 2 (CAMI2) short-read human microbiome toy datasets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Sczyrba, A. et al. Critical Assessment of Metagenome Interpretation—a benchmark of metagenomics software. Nat. Methods 14, 1063–1071 (2017)." href="/articles/s41587-026-03005-7#ref-CR37" id="ref-link-section-d164666590e718">37</a></sup>. Reassembly of CAMI2 was required because the original CAMI2 plasmids were not simulated as independent entities from their hosts cellular genomes and because assembly graphs were not available (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>). In our analysis, we explicitly consider plasmids and cellular genomes as separate entities that are binned independently. We found that PlasMAAG reconstructed 5–64% more near-complete (NC) bins and 10–57% more medium-quality (MQ) bins (≥0.9 precision, ≥0.5 recall) over all five benchmark datasets compared to VAMB, the second-best-performing binner on the benchmark data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2a</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1a</a>). The improvement in binning performance was driven by increased reconstruction of plasmids, where PlasMAAG reconstructed 50–121% (41–102%) more NC (MQ) plasmids candidate bins compared to SemiBin2, ComeBin, MetaBAT2, MetaDecoder, CONCOCT and VAMB across all benchmark datasets. When compared to SCAPP, PlasMAAG reconstructed 14–40% (17–45%) more NC (MQ) plasmids than SCAPP cycles over 4/5 (5/5) benchmark datasets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1b</a>). When evaluating confident plasmids bins generated by PlasMAAG (above the 0.95 geNomad plasmid threshold), PlasMAAG reconstructed 21–212% (50–171%) more NC (MQ) plasmid bins than SCAPP confident (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1c</a>). To check whether PlasMAAG’s accuracy stems only from its use of geNomad to identify plasmid contigs, we compared PlasMAAG to a workflow where geNomad was used to identify high-confidence plasmids in SCAPP cycles. However, PlasMAAG still outperformed this workflow, reconstructing 2–57% more NC plasmids and 6–44% more MQ plasmids across 4/5 and 5/5 benchmark datasets, respectively. (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">2</a>). Overall, these results show that PlasMAAG consistently outperforms existing binning tools in reconstructing plasmids.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="PlasMAAG binning and classification performance across the benchmark datasets."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: PlasMAAG binning and classification performance across the benchmark datasets.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="589"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, NC bins (cellular genomes + plasmids) reconstructed from the five benchmark datasets for VAMB, MetaBAT2, SemiBin2, MetaDecoder, ComeBin, SCAPP cycles, CONCOCT and PlasMAAG. <b>b</b>, NC plasmid bins reconstructed by all methods. <b>c</b>, NC plasmids reconstructed by SCAPP confident and PlasMAAG confident. <b>d</b>, Set of NC unique plasmids reconstructed only by PlasMAAG confident (red), only by SCAPP confident (green) and by both methods (light brown) across all datasets. <b>e</b>, Same as <b>d</b>, but for MQ plasmid bins. <b>f</b>, NC cellular bins reconstructed by all methods except SCAPP confident. <b>g</b>, Plasmid sample purity and recovery rate from the airways dataset for PlasMAAG across geNomad thresholds (dark red), PlasMAAG confident (dark-red star), geNomad across thresholds (green), geNomad at the default plasmid threshold (green cross), SCAPP cycles (light yellow) and SCAPP confident (dark yellow). <b>h</b>, Sample <i>F</i><sub>1</sub> across the five benchmark datasets for geNomad at the default plasmid threshold (green), SCAPP confident (yellow) and PlasMAAG confident (dark red). <b>i</b>, AUPRC for the classification of plasmids by geNomad (green) and when aggregating the geNomad scores per PlasMAAG community-based clusters (dark red). <b>j</b>, MCC for the classification of plasmids by geNomad (green) and when aggregating the geNomad scores per PlasMAAG community-based clusters (dark red).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM4">Source Data</a></p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec5">PlasMAAG provides a comprehensive and accurate characterization of microbial communities</h3><p>In addition to reconstructing more plasmids, the PlasMAAG confident set of plasmids also spanned a larger plasmid diversity, as the unique set of confident PlasMAAG plasmids across the benchmark datasets included 172 NC plasmid bins and 223 MQ plasmid bins not reconstructed by SCAPP confident. By contrast, SCAPP confident reconstructed 64 NC plasmid bins and 68 MQ plasmid bins not reconstructed by PlasMAAG. The intersecting set of plasmids reconstructed by both methods was 164 NC and 185 MQ plasmid bins, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2d,e</a>). PlasMAAG was also competitive at reconstructing cellular genomes, reconstructing only 0.7–12% fewer NC cellular bins than VAMB or CONCOCT, the best cellular binners on benchmark datasets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2f</a>), and 1–4% more MQ cellular bins over 3/5 benchmark datasets (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1d</a>). The set of PlasMAAG confident plasmids offered a better balance than SCAPP confident between the plasmid purity and recovery rates (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>) over the benchmark datasets, with a 14–43% improvement in <i>F</i><sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2g,h</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">3</a> and Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1</a>). This improvement was more pronounced in underexplored environments with fewer known plasmids, such as the CAMI2 marine dataset, where PlasMAAG achieved an <i>F</i><sub>1</sub> score 80% higher than SCAPP (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">4</a>). The PlasMAAG pipeline averages geNomad scores across sequences deemed likely to belong together improving classification compared to applying geNomad on individual contigs. This yielded an improvement in plasmid/nonplasmid contig classification area under the precision–recall curve (AUPRC) and Matthews correlation coefficient (MCC) of 28–69% and 42–131%, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig2">2i,j</a>, Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">2</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">5</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1</a>). By covering a broader range of plasmids, accurately reconstructing cellular genomes and achieving a better balance between plasmid purity and recovery rates, PlasMAAG provides a more comprehensive yet precise characterization of microbial communities.</p><h3 class="c-article__sub-heading" id="Sec6">AAGs integrate alignments and assembly graphs across samples</h3><p>Assembly graphs encode sequence overlaps between contigs known to provide valuable signals for binning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Lamurias, A., Sereika, M., Albertsen, M., Hose, K. &amp; Nielsen, T. D. Metagenomic binning with assembly graph embeddings. Bioinformatics 38, 4481–4487 (2022)." href="/articles/s41587-026-03005-7#ref-CR30" id="ref-link-section-d164666590e874">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Mallawaarachchi, V., Wickramarachchi, A. &amp; Lin, Y. GraphBin: refined binning of metagenomic contigs using assembly graphs. Bioinformatics 36, 3307–3313 (2020)." href="/articles/s41587-026-03005-7#ref-CR38" id="ref-link-section-d164666590e877">38</a></sup>. In this work, we enhanced assembly graphs to strengthen their binning signal in two steps. First, we applied an edge-weighting scheme to emphasize connections between taxonomically related contigs in assembly graphs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig3">3a,b</a>, Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">3</a>, and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">6</a>). Second, because assembly graphs only connect contigs from within a single sample, we aggregated them across samples by constructing an alignment graph, where edges link contigs on the basis of alignment information (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig3">3c,d</a> Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">8</a> and Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">4</a>). As assembly graphs edges and alignment graph edges are complementary, we could trivially merge the graphs by adding the edges from one graph into the other, thus creating the AAG. We evaluated the edges in the AAGs across the benchmark datasets to assess whether higher edge weights correspond to contigs that are taxonomically close, such as those from the same genome. We found that this was the case with AUPRCs of 0.69–0.90 (0.93–0.97) across the benchmark datasets at genome (species) taxonomic level (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig3">3f</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">9</a>). Consistent with this, we found a positive correlation between the averaged edge weights and the average edge accuracy at genome taxonomic level, with 0.20–0.97 Spearman correlation coefficients across benchmark datasets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig3">3g</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">9</a>). We then used fastnode2vec to project the AAG into numerical embeddings, from which we extracted communities of highly related contigs (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>). These AAG communities were precise but still represented only incomplete fragments of genomes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig3">3h</a>, Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">5</a>, upplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">10</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">2</a>). Thus, the AAG unifies assembly and alignment signals across samples into a graph whose edge weights capture taxonomic relationships, while fastnode2vec provides an efficient way to reduce this graph into embeddings that facilitate integration with other binning features.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Assembly graph, alignment graph and AAG-based features for binning."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Assembly graph, alignment graph and AAG-based features for binning.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="543"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Average precision of the assembly graph edges from the airways benchmark dataset, sorted by edge weight and bucketed into 20 nonoverlapping, equal intervals from 0 to 1, is shown for genome (blue), species (orange) and genus (green) taxonomic levels. Genome, species and genus Spearman correlation <i>P</i> values = 1.81 × 10<sup>−9</sup>, 1.06 × 10<sup>−6</sup> and 1.45 × 10<sup>−16</sup>, respectively (<i>X</i> = 20 buckets of edges). <b>b</b>, Precision–recall curve of the assembly graph edge weights from the airways benchmark dataset at genome (blue), species (orange) and genus (green) taxonomic levels. <b>c</b>, Alignment accuracy when increasing minimum identity thresholds across benchmark datasets. Results are shown only for restrictive alignments (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>) between contigs from different samples. <b>d</b>, Precision–recall curve of the alignment graph edge weights from the airways benchmark dataset at genome (blue), species (orange) and genus (green) taxonomic levels. <b>e</b>, Average precision of the alignment graph edges from the airways benchmark dataset, sorted by weight and grouped into 5% bins, is shown for genome (blue), species (orange) and genus (green) taxonomic levels. Genome, species and genus Spearman correlation <i>P</i> values = 2.1 × 10<sup>−</sup><sup>6</sup>, 3.94 × 10<sup>−</sup><sup>6</sup> and 3 × 10<sup>−</sup><sup>5</sup>, respectively (<i>X</i> = 20 bins of edges). <b>f</b>. Precision–recall curve of the AAG edge weights from the airways benchmark dataset at genome (blue), species (orange) and genus (green) taxonomic levels. <b>g</b>, Average precision of AAG edges from the airways benchmark dataset, sorted by weight and grouped into 5% bins, is shown for genome (blue), species (orange) and genus (green) taxonomic levels. Genome, species and genus Spearman correlation <i>P</i> values = 1.09 × 10<sup>−</sup><sup>12</sup>, 1.38 × 10<sup>−</sup><sup>15</sup> and 9.58 × 10<sup>−</sup><sup>19</sup>, respectively (<i>X</i> = 20 bins of edges). <b>h</b>, Precision distribution of communities extracted using fastnode2vec from AAGs across the five benchmark datasets at the genome taxonomic level. </p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM4">Source Data</a></p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec7">Contrastive VAEs improve binning through aggregating, merging and splitting communities</h3><p>To address genome fragmentation in the AAG communities, we leveraged traditional binning features such as contig <i>k</i>-mer composition and abundances<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Mallawaarachchi, V. et al. Solving genomic puzzles: computational methods for metagenomic binning. Brief. Bioinform. 25, bbae372 (2024)." href="/articles/s41587-026-03005-7#ref-CR39" id="ref-link-section-d164666590e1040">39</a></sup>. In the VAMB framework, contig features are embedded using a VAE and these embeddings are then used for clustering contigs. PlasMAAG follows the same approach but also considers community structure during the embedding and clustering process. To encourage contigs of the same community to be close in the embedding while allowing contigs from different communities to be close in latent space when supported by other signals, we modified the VAE loss function. Specifically, we added a contrastive loss term that penalized large embedding distance between contigs of the same community. We then applied a clustering strategy on the embeddings, consisting of three key steps: (1) merging, where communities close in the embedding were merged to reduce genome fragmentation; (2) splitting, where communities with contigs placed far apart in the embedding were split up to increase precision; (3) expansion, where unassigned contigs located close to a community in latent space were added to the community to improve recall. We refer to these three steps as ‘community-based’ clustering (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">11</a>). This community-based clustering resulted in a 46–102% increase in genome recall across benchmark datasets compared to the raw communities, confirming the effectiveness of these steps. The splitting step improved precision by 0.03–1% (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">12</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">3</a>), indicating minor but positive impact without compromising recall. On the other hand, community expansion had limited effect, with only a 1–3% increase in community size (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">3</a>), suggesting that step 3 had a smaller impact. As recall increased and precision slightly improved, <i>F</i><sub>1</sub> scores also increased, along with the number of reconstructed NC bins. Additionally, we evaluated the impact of including the contrastive loss term into the VAE on binning when clustering the embeddings with either the VAMB original density-based clustering or the community-based clustering. Across datasets, the contrastive loss substantially increased <i>F</i><sub>1</sub> scores and had an overall positive effect on the number of NC bins (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">6</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">13</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">14</a>). Together, these results indicate that genome reconstruction benefited from latent representations informed by AAG-derived communities.</p><h3 class="c-article__sub-heading" id="Sec8">Plasmids and cellular genomes require distinct clustering strategies</h3><p>We found that cellular genomes, likely because of their larger sizes, were fragmented across more AAG-derived communities compared to plasmids. In addition, plasmids and cellular genomes showed distinct patterns in <i>k</i>-mer composition, contig coabundance and PlasMAAG latent representations. Together, these differences suggested that community-based clustering was more appropriate for plasmid binning, whereas density-based clustering was more appropriate for cellular genomes (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">7</a>, Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">15</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">17</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">4</a>). Because community-based clustering relied on AAG-derived features, we further explored how sample number and intersample similarity affected PlasMAAG’s performance. We found that larger and homogeneous sample sets led to improved binning accuracy and sample characterization (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">8</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">18</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">19</a>). Therefore, to identify and extract potential plasmids from the communities, we used geNomad to assign plasmid scores to each community-based cluster<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Camargo, A. P. et al. Identification of mobile genetic elements with geNomad. Nat. Biotechnol. 42, 1303–1312 (2024)." href="/articles/s41587-026-03005-7#ref-CR36" id="ref-link-section-d164666590e1114">36</a></sup>. We found that averaging geNomad scores across communities led to more accurate plasmid identification than scoring individual contigs (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">2</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">7</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">1</a>). This enabled us to extract communities as putative plasmid bins with the community-based clustering and to remove plasmid contigs from the density-based clusters. We further observed that the plasmid and cellular extraction was sensitive to the geNomad threshold, particularly for cellular genomes (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">9</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">20</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">21</a>). For example, applying a geNomad plasmid threshold of 0.7 resulted in a reduction in NC cellular genomes (plasmids) of 6–39% (3–18%) (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">22</a>). This indicated a trade-off between plasmid extraction and cellular genome recovery. Importantly, in datasets where we added phages to plasmids and chromosomes, our approach remained robust, leading to reduced viral contamination while maintaining comparable plasmid reconstruction performance (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">10</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">23</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">5</a>). Lastly, we evaluated whether PlasMAAG’s plasmid reconstruction exhibited biases related to biological features such as gene content, copy number or plasmid length. Across the benchmark datasets, we observed no clear bias toward plasmids carrying transposons, insertion sequences or antibiotic resistance genes nor toward specific plasmid copy number intervals or plasmid lengths below 350 kb. However, we did find that reconstruction rates were lower for very large plasmids exceeding 500 kb (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">11</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">24</a>). Together, these results demonstrate that explicitly accounting for the different community structure of plasmids and cellular genomes enables robust plasmid reconstruction while balancing plasmid and cellular genome recovery.</p><h3 class="c-article__sub-heading" id="Sec9">Validating PlasMAAG plasmid binning using paired short-read and long-read samples</h3><p>Validating PlasMAAG binning performance on real data is challenging because existing tools do not provide quality estimates for plasmids and may be biased in poorly characterized environments. To address this, we designed a validation strategy based on five sewage samples from Denmark sequenced both with short-read and long-read sequencing. This paired dataset enabled us to use long-read contigs to evaluate bins generated from the short-read assemblies (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig4">4a</a>). Using this approach, PlasMAAG community-based bins reconstructed 117% more long-read contigs than VAMB, the second-best-performing binner (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig4">4b</a>). This improved performance remained consistent even after accounting for incompleteness in the long-read assembled contigs (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">12</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">25</a>). To further assess plasmid binning performance, we additionally evaluated reconstruction against plasmid-enriched sequencing data of the five sewage samples (plasmidomics)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Yu, Z. et al. Insights into the circular: the cryptic plasmidome and its derived antibiotic resistome in the urban water systems. Environ. Int. 183, 108351 (2024)." href="/articles/s41587-026-03005-7#ref-CR40" id="ref-link-section-d164666590e1176">40</a></sup>. Here, PlasMAAG reconstructed 33 NC long-read plasmid contigs, corresponding to 22% more than the second best binner VAMB and 136% more NC plasmids than SCAPP cycles (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig4">4b</a>). However, identification of plasmid bin candidates remained challenging, as geNomad assigned low plasmid scores to many long-read contigs supported by the plasmidomics data, suggesting that a considerable diversity of plasmids remains to be discovered. On the other hand, cellular genome reconstruction was unaffected by geNomad filtering, with PlasMAAG showing competitive cellular genome binning performance (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig4">4b–d</a>, Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">21</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">26</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">29</a> and Supplementary Notes <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">13</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">14</a>). Together, these results demonstrate that PlasMAAG delivered superior binning performance on real datasets, particularly for plasmid sequences, while highlighting that geNomad-based plasmid identification should be used with caution.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="PlasMAAG on real samples from hospital sewage."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: PlasMAAG on real samples from hospital sewage.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="588"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Overview of the strategy used to validate PlasMAAG on the five hospital sewage samples. For each sample, long-read metagenomics, short-read metagenomics and short-read plasmidomics datasets were generated. LR, long-read; SR, short-read. PlasMAAG was applied to the short-read metagenomics data to produce candidate plasmid and cellular bins. These bins were validated against a reference assembly composed by long-read contigs to assess overall binning performance and against a second reference assembly constructed from long-read contigs with plasmid evidence, identified by either circularity or plasmidomics read coverage. <b>b</b>, Binning performance of all methods across the five sewage samples, evaluated using all long-read contigs (red) and long-read contigs with plasmid evidence (blue). PlasMAAG dens., bins produced using VAMB’s density-based clustering algorithm on PlasMAAG’s latents; PlasMAAG coms., bins generated using the community-based clustering algorithm. <b>c</b>, Binning performance of PlasMAAG, SCAPP and MetaPlasmidSPAdes under relaxed (light gray) and strict (dark gray) plasmid filtering criteria. <b>d</b>, NC cellular bins according to CheckM2 estimates, produced by all organism binners for the five hospital sewage samples. PlasMAAG non-plas. conf., PlasMAAG density-based bins after extracting candidate plasmid contigs by aggregating geNomad plasmid contig scores per PlasMAAG community-based clusters (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>). </p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM4">Source Data</a></p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec10">Identification of host–plasmid associations from hospital sewage environments</h3><p>By reconstructing plasmid and cellular genomes from the same set of samples, PlasMAAG enables integrated host–plasmid analyses. To illustrate this, we investigated host–plasmid abundance correlations of 24 short-read-based hospital sewage samples collected in Spain (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>). PlasMAAG reconstructed 27,954 candidate plasmid bins and 213,431 nonplasmid candidate bins. Leveraging the multisample design, plasmid bins were aggregated into 13,912 cross-sample clusters and putative bacterial hosts were assigned to each plasmid cluster using the Plasmid Database (PLSDB). In parallel, we identified 323 high-quality cellular organism bins (completeness ≥ 70%, contamination ≤ 10%) and, as with the plasmids, aggregated them across the samples. Using abundance information across samples, we identified a total of 825 significant positive correlations between plasmid and cellular clusters (adjusted <i>P</i> value &lt; 0.05, two-sided test) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5a</a>). Of these, 209 correlations were consistent with known host–plasmid associations at the phylum level according to PLSDB, 237 were inconsistent (involving plasmids known to associate with different phyla) and the remaining 379 could not be evaluated because they involved plasmid sequences with no match in PLSDB (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">30</a>). Some host–plasmid associations matched known and unknown patterns. For instance, cluster cl_20, annotated as belonging to the <i>Aeromonas</i> genus, correlated with up to 41 plasmid clusters, 12 of which were previously reported as known host–plasmid associations in PLSDB (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5a</a>). On the other hand, cluster cl_293, annotated as the <i>Ruminococcus_E</i> genus, correlated with 43 plasmid clusters, none of them previously reported in PLSDB (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5a</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="PlasMAAG enables host–plasmid association studies and exploration of intraplasmid variation across environments, demonstrated using 24 hospital sewage samples."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: PlasMAAG enables host–plasmid association studies and exploration of intraplasmid variation across environments, demonstrated using 24 hospital sewage samples.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="569"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Spearman correlation between PlasMAAG high-quality cellular clusters and PlasMAAG plasmid clusters with an aggregated geNomad plasmid score above 0.75. Highlighted cells with bold rectangles indicate significant correlations after Benjamini–Hochberg false discovery rate correction. Cells marked with ‘X’ represent plasmid–organism associations previously reported in PLSDB. The organism cluster dendrogram was generated using GTDB-tk taxonomic annotations, while the plasmid cluster dendrogram was based on abundance correlations. <b>b</b>, PlasMAAG plasmid cluster nneighs_416 bins. Each row represents a bin from one sample and numbers within parenthesis indicate median bin depth. Yellow blocks denote contigs aligned to <i>pAsa4c</i>, sorted by alignment position. Dark-green blocks represent contigs not mapping to <i>pAsa4c</i> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec13">Methods</a>), with their positions inferred from matches to other PLSDB plasmid accessions. Light-green sections withing dark-green blocks indicate alignment segments to <i>pAsa4c</i>. Dark-gray areas indicate alignment graph edges and light-gray areas represent nonrestrictive alignment matches. GC%, average G+C content computed using a 1,000-kb window. Color code for <i>pAsa4c</i> regions: blue, replication and maintenance; green, conjugative transfer; purple, recombination and DNA repair; orange, secretion and surface structures; red, metabolism; yellow, enzymes; cyan, regulatory proteins and transcription factors; brown, transposases and mobile genetic elements; gray, hypothetical or unclassified. <b>c</b>, PlasMAAG plasmid cluster nneighs_416, composed of contigs from sample 6 (blue), sample 3 (red), sample 5 (green) and sample 23 (orange). Links represent alignment regions, colored according to the sample of origin. Bold annotations indicate functions associated to antimicrobial resistance. </p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM4">Source Data</a></p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41587-026-03005-7/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec11">Studying intraplasmid variation across hospital sewage samples</h3><p>In PlasMAAG, contigs from different samples are projected into a shared latent space, enabling them to be clustered together and split into per-sample bins thereafter. Analysis of these cross-sample clusters enabled investigation of highly related plasmids from different samples and bins from the same plasmid cluster could be explored using the alignments from the AAG. To illustrate this, we selected plasmid cluster nneighs_76 for a more in-depth analysis. Plasmid bins from the nneighs_76 cluster reconstructed a 90-kb region similar to the plasmid <i>pAsa4c</i>, which is reported to be hosted by <i>Aeromonas</i> <i>salmonicida</i> subsp. <i>salmonicida</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Tanaka, K. H. et al. The mosaic architecture of Aeromonas salmonicida subsp. salmonicida pAsa4 plasmid and its consequences on antibiotic resistance. PeerJ 4, e2595 (2016)." href="/articles/s41587-026-03005-7#ref-CR41" id="ref-link-section-d164666590e1341">41</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5b</a>). Despite representing highly overlapping regions of the same accession, sample-specific bins from the nneighs_76 cluster exhibited varying degrees of contig fragmentation. For instance, the bin from sample 2 was composed of 20 contigs, whereas the bin from sample 1 consisted of ten contigs, which could be explained by the difference in the plasmid abundance in different samples. We then explored the relationship of the plasmid bins using the alignments from the AAG and found that some bins in nneighs_76 contained contigs that did not align to <i>pAsa4c</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5b</a>). Several of these unaligned contigs were found in multiple bins and were syntenic across bins aligned to each other, suggesting that we found true plasmid variation and not an error in binning (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5b</a>). Using synteny and the AAG, we could identify four approximate locations in the reference sequence where these contigs could be located. Three of four regions had hallmarks of recombination hotspots, including an ISAs2 insertion site, a known conjugative block and a segment with distinct G+C content<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Tanaka, K. H. et al. The mosaic architecture of Aeromonas salmonicida subsp. salmonicida pAsa4 plasmid and its consequences on antibiotic resistance. PeerJ 4, e2595 (2016)." href="/articles/s41587-026-03005-7#ref-CR41" id="ref-link-section-d164666590e1358">41</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5b</a>). Furthermore, 14 of 19 contigs not mapping to <i>pAsa4c</i> aligned to plasmid accessions reported to be hosted by organisms from the <i>Aeromonas</i> genus, consistent with the <i>pAsa4c</i> original host and supporting the findings. Additionally, PlasMAAG clusters, in combination with the AAG, enabled the exploration of diversity among similar plasmids across samples without PLSDB support. As an example, sample-specific bins from the plasmid cluster nneighs_416 exhibited a high degree of sequence similarity despite variations in contig fragmentation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41587-026-03005-7#Fig5">5c</a>). In conclusion, PlasMAAG facilitates the tracking of highly similar plasmids across different environments, allowing for the capture of their composition variations.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec12-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec12">Discussion</h2><div class="c-article-section__content" id="Sec12-content"><p>Plasmids are pivotal in horizontal gene transfer, with an influential role in shaping microbial communities. Their prevalence across microbial ecosystems highlights their importance, yet studying plasmids from environmental samples has been challenging because of their dynamic and unstable composition, as evidenced by the lack of groundbreaking plasmid binners in recent years. This limitation has hindered efforts to bin and identify plasmids accurately, despite their abundance. The decrease in sequencing costs has greatly increased the availability of metagenomic samples, presenting an unprecedented opportunity to uncover plasmid diversity. However, the challenges of plasmid binning emphasize the need for a robust and broad-range plasmid binning method.</p><p>In this study, we introduced PlasMAAG, a deep-learning-based method for metagenomic binning of both plasmids and cellular genomes. PlasMAAG uses a unique feature we developed, AAGs, which enables the aggregation of assembly graphs across multiple samples. This advancement allows PlasMAAG to overcome traditional limitations associated with single-sample plasmid assemblers.</p><p>PlasMAAG outperformed SCAPP, the current state-of-the-art plasmid assembler, on both synthetic and real datasets, delivering superior results for plasmid binning while being substantially faster. In addition to producing more plasmid bins, the set of candidate plasmids produced by PlasMAAG achieved a more balanced trade-off between plasmid purity and recovery rates, enabling a broader characterization of metagenomic samples. However, it is important to note that SCAPP is not a binner but a plasmid assembler focused on generating complete circular contigs. As such, its goal is different from that of PlasMAAG, which aims to bin plasmid fragments into coherent groups.</p><p>PlasMAAG’s capability to bin both plasmids and organisms offers a comprehensive approach to metagenomic analysis. PlasMAAG achieves organism binning results that are comparable to leading organism binners on synthetic datasets and real-world environmental datasets. PlasMAAG’s holistic approach enables integrated studies, such as the exploration of plasmid–host associations. Using its comprehensive binning capabilities, we gathered abundance-correlation-based evidence for 825 plasmid–host associations, with only 4% previously reported in PLSDB. However, these associations are merely based on abundance correlation; thus, we expect a substantial fraction of them to be spurious. Studies of host–plasmid associations could use PlasMAAG correlations as a starting point but would need more evidence. By contrast, PlasMAAG’s AAG-based clustering robustly revealed intraplasmid variation across samples, enabling the study of plasmid sequence variation across environments. Nevertheless, we expect PlasMAAG to be poor at recovering within-sample intraplasmid variation because of limited preservation of microdiversity by short-read assemblers such as metaSPAdes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Nurk, S., Meleshko, D., Korobeynikov, A. &amp; Pevzner, P. A. metaSPAdes: a new versatile metagenomic assembler. Genome Res. 27, 824–834 (2017)." href="/articles/s41587-026-03005-7#ref-CR42" id="ref-link-section-d164666590e1396">42</a></sup> and difficulty in accurately estimating abundance of many near-identical contigs within a single sample.</p><p>We demonstrated that geNomad plasmid predictions were enhanced when aggregated across PlasMAAG communities, underscoring the value of binning for refining plasmid sequence identification. However, we also found that geNomad was inaccurate when applied to understudied environments, as validated by experimental paired metaplasmidomics, long-read and short-read data. These discrepancies highlight the need for more robust plasmid sequence identification methods capable of handling complex or uncharted environments.</p><p>The success of PlasMAAG is largely attributable to the AAG, a feature that complements assembly graph signals across samples in a multisample framework. This innovation not only enhances binning accuracy but also facilitates the inference of compositional similarities between samples. Moreover, AAGs also improve binning of contigs from the same sample by indirectly linking within-sample contigs through between-sample intermediaries.</p><p>Another notable innovation in PlasMAAG is its use of contrastive loss to integrate traditional binning features, such as <i>k</i>-mer composition and contig abundances, with the AAG. This approach could be extended to incorporate other graph-like data in the binning process, such as Hi-C data. As sequencing technologies advance and contigs become decreasingly fragmented, particularly in long-read datasets, the utility of using cross-sample alignments to bridge gaps in the assembly graphs will grow, covering larger genome fractions and providing richer insights.</p><p>By enabling the study of plasmids alongside cellular genomes from highly complex samples, PlasMAAG expands our ability to explore microbial communities comprehensively. Its focus on plasmids, an often-overlooked but critical component of microbial ecosystems, enhances our understanding of their role in horizontal gene transfer and microbial community dynamics.</p><p>In conclusion, PlasMAAG represents a step forward in plasmid and organism binning from metagenomic samples. By incorporating AAGs and contrastive learning, it addresses longstanding challenges in plasmid binning while providing a framework for studying plasmid–host associations and microbial community dynamics.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec13-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec13">Methods</h2><div class="c-article-section__content" id="Sec13-content"><h3 class="c-article__sub-heading" id="Sec14">Overview of PlasMAAG</h3><p>The inputs to the PlasMAAG pipeline are a set of reads per sample. Reads are assembled with metaSPAdes (version 3.15.5)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Nurk, S., Meleshko, D., Korobeynikov, A. &amp; Pevzner, P. A. metaSPAdes: a new versatile metagenomic assembler. Genome Res. 27, 824–834 (2017)." href="/articles/s41587-026-03005-7#ref-CR42" id="ref-link-section-d164666590e1431">42</a></sup>, creating an assembly graph and contigs for each sample. The contigs across all samples are concatenated together to create the contig catalog. Reads are mapped to the catalog with minimap2 (version 2.24)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Li, H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics 34, 3094–3100 (2018)." href="/articles/s41587-026-03005-7#ref-CR43" id="ref-link-section-d164666590e1435">43</a></sup> and SAMtools (version 1.18)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079 (2009)." href="/articles/s41587-026-03005-7#ref-CR44" id="ref-link-section-d164666590e1439">44</a></sup>, creating per-sample BAM files. The alignment graph is generated by aligning the contigs across samples with the National Center for Biotechnology Information (NCBI) basic local alignment search tool (BLAST; version 2.15.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. J. Mol. Biol. 215, 403–410 (1990)." href="/articles/s41587-026-03005-7#ref-CR45" id="ref-link-section-d164666590e1443">45</a></sup>. The assembly and alignment graphs are merged into the AAG. Then, fastnode2vec (version 0.05)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Abraham, L. louisabraham/fastnode2vec version-0.0.5 Zenodo &#xA;                https://doi.org/10.5281/zenodo.3902632&#xA;                &#xA;               (2020)." href="/articles/s41587-026-03005-7#ref-CR35" id="ref-link-section-d164666590e1447">35</a></sup>, an optimized version of node2vec<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Grover, A. &amp; Leskovec, J. Node2vec: scalable feature learning for networks. Preprint at &#xA;                https://doi.org/10.48550/arXiv.1607.00653&#xA;                &#xA;               (2016)." href="/articles/s41587-026-03005-7#ref-CR46" id="ref-link-section-d164666590e1452">46</a></sup>, is used to embed local AAG context of each contig into an embedding space. The <i>k</i>-mer composition and abundance features of contigs are embedding using a VAE, where an additional loss term is added, which penalizes distance between contigs of the same community. Using the VAE embeddings, communities are expanded, merged and purified. The geNomad tool (version 1.8.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Camargo, A. P. et al. Identification of mobile genetic elements with geNomad. Nat. Biotechnol. 42, 1303–1312 (2024)." href="/articles/s41587-026-03005-7#ref-CR36" id="ref-link-section-d164666590e1459">36</a></sup> is used to separate plasmid from nonplasmid contigs; communities of plasmid contigs are extracted as separate bins, whereas the remaining contigs are extracted in bins using a clustering algorithm.</p><h3 class="c-article__sub-heading" id="Sec15">Benchmark datasets</h3><p>We based our benchmark datasets on the existing CAMI2 human microbiome toy dataset, the CAMI2 marine dataset and the CAMI2 skin human microbiome dataset with addition of phage reads from 280 phage genomes from the Metagenomic Gut Virus (MGV) database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Nayfach, S. et al. Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome. Nat. Microbiol. 6, 960–970 (2021)." href="/articles/s41587-026-03005-7#ref-CR47" id="ref-link-section-d164666590e1471">47</a></sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">5</a>). All CAMI2 datasets are publicly available short-read metagenomic benchmarks generated under the CAMI initiatives<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Sczyrba, A. et al. Critical Assessment of Metagenome Interpretation—a benchmark of metagenomics software. Nat. Methods 14, 1063–1071 (2017)." href="/articles/s41587-026-03005-7#ref-CR37" id="ref-link-section-d164666590e1478">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Meyer, F. et al. Critical Assessment of Metagenome Interpretation: the second round of challenges. Nat. Methods 19, 429–440 (2022)." href="/articles/s41587-026-03005-7#ref-CR48" id="ref-link-section-d164666590e1481">48</a></sup>. The human dataset consists of five simulated environments: airways, gastrointestinal, oral, skin and urogenital, designed to mimic human microbiome communities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Sczyrba, A. et al. Critical Assessment of Metagenome Interpretation—a benchmark of metagenomics software. Nat. Methods 14, 1063–1071 (2017)." href="/articles/s41587-026-03005-7#ref-CR37" id="ref-link-section-d164666590e1485">37</a></sup>, whereas the CAMI2 marine dataset represents a sea environment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Meyer, F. et al. Critical Assessment of Metagenome Interpretation: the second round of challenges. Nat. Methods 19, 429–440 (2022)." href="/articles/s41587-026-03005-7#ref-CR48" id="ref-link-section-d164666590e1489">48</a></sup>. For each sample, we simulated 5 Gbp of 2× 150-bp Illumina-like reads from the source genomes using the CAMISIM software<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Fritz, A. et al. CAMISIM: simulating metagenomes and microbial communities. Microbiome 7, 17 (2019)." href="/articles/s41587-026-03005-7#ref-CR49" id="ref-link-section-d164666590e1494">49</a></sup>. Compared to the original CAMI2 datasets, we had to make several changes. The original dataset did not provide assembly graphs; thus, we assembled the reads using metaSPAdes (version 3.15.5) and mapped the resulting contigs back to the CAMI2 source genomes or the MGV phage genomes to determine their origin using minimap2, accepting hits with an identity &gt; 97% and a query coverage &gt; 90%. Because this approach initially led to many unmapped or ambiguously mapping contigs because of overeager read correction in the assembler, we resimulated the reads using wgsim (<a href="https://github.com/lh3/wgsim">https://github.com/lh3/wgsim</a>) with zero sequencing errors and a default fragment size of 270 (mean)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Wu, Y.-W., Simmons, B. A. &amp; Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. Bioinformatics 32, 605–607 (2016)." href="/articles/s41587-026-03005-7#ref-CR26" id="ref-link-section-d164666590e1505">26</a></sup> and disabled error correction in the assembler. Moreover, CAMI2 considered plasmids to be part of their cellular host genome with the same abundance, which would inhibit our abundance-based binning approach. Instead, we simulated plasmids as distinct genomes with their own abundance proportional to the host abundance, modulated by a Gaussian random variable that incorporates a plasmid copy number model, as performed previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Pellow, D. et al. SCAPP: an algorithm for improved plasmid assembly in metagenomes. Microbiome 9, 144 (2021)." href="/articles/s41587-026-03005-7#ref-CR17" id="ref-link-section-d164666590e1509">17</a></sup>. For the samples containing phages, phage abundances were sampled from a lognormal distribution with mean of 1 and s.d. of 4. Lastly, CAMI2 did not contain reads simulated from across the edges of the underlying circular sequences, which prevents assembly graph cycles and hobbles graph peeling-based approaches such as that used by SCAPP. We made sure to include such reads and enable circular contigs by simulating reads spanning the junction between the start and end positions of the genomes. Our reassembled CAMI2 human and marine datasets were, like the original CAMI2 datasets, composed of ten samples for all datasets except urogenital, which had nine. Our datasets had the following number of contigs after retaining only contigs larger than 2 kbp: airways, 35,537; gastrointestinal, 80,325; oral, 94,251; skin, 31,181; urogenital, 29,550; marine, 150,565; skin with phages, 34,979.</p><h3 class="c-article__sub-heading" id="Sec16">Assembly graph edge weighting</h3><p>Assembly graphs were extracted from the assembly_graph_after_simplification.gfa file from metaSPAdes and converted into a NetworkX (version 3.4.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Hagberg, D. &amp; Pieter, J. Exploring network structure, dynamics, function using NetworkX. In Proc. 24th Science in Python Conference (eds Ambade, A. et al.) (SciPy, 2025)." href="/articles/s41587-026-03005-7#ref-CR50" id="ref-link-section-d164666590e1521">50</a></sup> directed graph, with contigs represented as nodes and links between segments in contigs represented as edges. To enrich the assembly graph signal for binning, graph edges were weighted with the normalized linkage metric, which is dependent on the number of links between any segments from each pair of contigs, normalized by the length of the contigs. For a pair of contigs <i>c</i><sup><i>i</i></sup>, <i>c</i><sup><i>j</i></sup>, the number of links connecting those contigs <i>n</i>_links<sub><i>ij</i></sub> and the contig lengths <i>l</i><sup><i>c</i></sup>, normalized linkage is expressed as</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\mathrm{normalized}\,{\mathrm{linkage}}_{{c}^{i}{c}^{j}}=\frac{{n\mathrm{\_links}}_{{ij}}}{\min ({l}^{{c}^{i}},{l}^{{c}^{j}})}$$</span></div></div><h3 class="c-article__sub-heading" id="Sec17">Alignment graph edge weighting</h3><p>After assembly, contigs shorter than 2,000 bp were discarded, as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Nissen, J. N. et al. Improved metagenome binning and assembly using deep variational autoencoders. Nat. Biotechnol. 39, 555–560 (2021)." href="/articles/s41587-026-03005-7#ref-CR24" id="ref-link-section-d164666590e1675">24</a></sup>. Contigs were aligned all against all using NCBI BLAST using the ‘blastn’ command with ‘-perc_identity 95’, only keeping between-sample hits, alignment identity ≥ 98.0% and alignment ≥ 500 bp. We also removed alignments between sequences that contained large sections that did not align because of sequence diversity, as we wanted the alignments to represent shared sequences across samples. The remaining set of alignments after filtering was defined as ‘restrictive’ alignments. From the alignments, we created an alignment graph with contigs as nodes and alignments as edges. Edges were weighted with the normalized alignment metric to reflect the alignment certainty. For a pair of contigs <i>c</i><sup><i>i</i></sup>, <i>c</i><sup><i>j</i></sup>, alignment identity id, alignment length <i>L</i> and contig length <i>l</i><sup><i>c</i></sup>, normalized alignment is expressed as</p><div id="Equb" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\mathrm{normalized}\,{\mathrm{alignment}}_{{c}^{i}{c}^{j}}=\frac{\mathrm{id}}{100}\frac{\min (L,{l}^{{c}^{i}},{l}^{{c}^{j}})}{\min ({l}^{{c}^{i}},{l}^{{c}^{j}})}$$</span></div></div><h3 class="c-article__sub-heading" id="Sec18">AAG community extraction with fastnode2vec</h3><p>Assembly and alignment graphs share no edges because their edges connect only within-sample and between-sample contigs, respectively. This allowed us to trivially merge the graphs by adding the edges from one graph into the other, thus creating the AAG. Thus, the AAG is a single graph that contains the nodes and edges from the individual assembly graphs and the alignment graph. We aimed to leverage this integrated graph to enhance the binning signal, while keeping all individual nodes (that is, contigs) intact and unchanged. To extract communities from the AAG, we first ran fastnode2vec on the AAG to obtain contig embeddings. We created a new graph by linking contigs within a cosine distance of 0.1 in embedding space, after which we defined each connected component to be a contig community. We optimized the fastnode2vec hyperparameters and clustering radius to generate pure communities at the genome level, running a small grid search over the resimulated CAMI2 airways dataset. The embedding dimensions, walk length, number of walks, window size, <i>p</i> and <i>q</i> parameters from fastnode2vec were set to 32, 10, 50, 10, 0.1 and 2.0, respectively.</p><h3 class="c-article__sub-heading" id="Sec19">Contrastive-VAMB for community merging and expansion</h3><p>Contrastive-VAMB is a variation of the original VAMB model, with a modification on the loss function to account for the communities extracted from the fastnode2vec embeddings. Contrastive-VAMB is composed of an encoder, latent representation layer μ and a decoder. Each contig represented by the concatenation of the contig coabundances along samples <b>C</b><sub>in</sub>, the tetranucleotide frequencies <b>T</b><sub>in</sub> and the unnormalized contig abundances <b>A</b><sub>in</sub> is passed to the encoder. While <b>C</b><sub>in</sub> and <b>T</b><sub>in</sub> were already used in the original VAMB model, the inclusion of <b>A</b><sub>in</sub> is motivated by the observation that the normalization in the computation of <b>C</b><sub>in</sub> removes abundance signal by removing a degree of freedom. This signal is reintroduced as <b>A</b><sub>in</sub>. This way, the network explicitly models two distinct notions of abundance: how much abundance there is in total and how it is distributed among samples. The encoder projects the contigs into a latent normal N(μ, I) distribution parametrized by the μ layer, from which the decoder samples. The decoder is optimized to reconstruct <b>C</b><sub>in</sub>, <b>T</b><sub>in</sub> and <b>A</b><sub>in</sub> from the instances sampled from N(μ, I), decrease the latent cosine distance between contigs with closely related fastnode2vec graph embeddings and decrease the deviance between the latent normal distribution N(μ, I) parametrized by the μ layer and the standard normal distribution used as a prior N(0, I).</p><h3 class="c-article__sub-heading" id="Sec20">Loss functions</h3><p>The contrastive-VAMB loss can be decomposed in three terms: reconstruction loss, contrastive loss and regularization loss. The reconstruction loss (<i>L</i><sub>rec</sub>) penalizes the reconstruction error of <b>A</b><sub>in</sub>, <b>T</b><sub>in</sub>, and <b>C</b><sub>in</sub>. Like in VAMB, the sum of squared error (SSE) loss was used for <b>T</b><sub>in</sub> and cross-entropy (CE) loss was used for <b>C</b><sub>in</sub>. For <b>A</b><sub>in</sub>, PlasMAAG also uses SSE loss. These three terms are weighted with hyperparameters <i>w</i><sub><b>A</b></sub>, <i>w</i><sub><b>T</b></sub> and <i>w</i><sub><b>C</b></sub>.</p><div id="Equc" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${L}_{\mathrm{rec}}={w}_{{\bf{A}}}\mathrm{CE}\left({{\bf{A}}}_{\mathrm{in}},{{\bf{A}}}_{\mathrm{out}}\right)+{w}_{{\bf{T}}}\mathrm{SSE}\left({{\bf{T}}}_{\mathrm{in}},{{\bf{T}}}_{\mathrm{out}}\right)+{w}_{{\bf{C}}}\mathrm{SSE}\left({{\bf{C}}}_{\mathrm{in}},{{\bf{C}}}_{\mathrm{out}}\right)$$</span></div></div><p>The contrastive loss (<i>L</i><sub>contr</sub>) penalizes the cosine distance between the VAMB-latent representations contigs highly related in fastnode2vec embedding space when such cosine distance overcomes a predefined margin <i>m</i>, where <i>m</i> is a hyperparameter. For a contig <i>ci</i> and highly related fastnode2vec embedding space contigs <i>H</i><sup><i>ci</i></sup> = {<i>n</i><sub>0,…,</sub> <i>n</i><sub><i>n</i></sub>}, <i>L</i><sub>contr</sub> is expressed as</p><div id="Equd" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${L}_{\mathrm{contr}}=\max \left(\frac{{\sum }_{{n}_{i}\in {H}^{ci}}\mathrm{cosine}\,\mathrm{distance}\left({{\rm{m}}}^{ci},{{\rm{m}}}^{{n}_{i}}\right)}{\left|{H}^{ci}\right|}-m,0\right)$$</span></div></div><p>The regularization loss (<i>L</i><sub>reg</sub>) penalizes the deviance between the latent normal distribution N(μ, I) parametrized by the μ layer and the standard normal distribution used as a prior N(0, I) with the Kullback–Leibler divergence; as the s.d. is set to 1, <i>L</i><sub>reg</sub> can be simplified to</p><div id="Eque" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${L}_{\mathrm{reg}}=\frac{1}{2}+\sum {{\rm{\mu }}}^{2}$$</span></div></div><p>Lastly, the model total loss (<i>L</i>) was aggregated with weighting hyperparameters <i>w</i><sub><i>L</i>reg</sub>, and <i>w</i><sub><i>L</i>contr</sub>:</p><div id="Equf" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$L={L}_{\mathrm{rec}}+{{w}_{{L}_{\mathrm{reg}}}L}_{\mathrm{reg}}+{{w}_{{L}_{\mathrm{contr}}}+L}_{\mathrm{contr}}$$</span></div></div><h3 class="c-article__sub-heading" id="Sec21">Clustering plasmid and organism candidates with geNomad</h3><p>Two sequential strategies were implemented to cluster the latent space tailored to extract plasmids and nonplasmids. The clustering strategy is composed of two phases: community clustering and iterative medoid clustering, both using latent space cosine distances. Community clustering works in five steps (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">11</a>). First, for each community extracted from the fastnode2vec embeddings, contigs belonging to the same community are linked and links between contigs with a VAE embedding cosine distance &gt; 0.2 are removed. Second, contigs within 0.01 cosine distance are linked. Third, connected components are extracted as bins. Fourth, potentially circular contigs are detected by mapping read pairs, where mates map to opposite contig ends within 50 bp of the contig end. These are extracted to their own bins. Lastly, plasmid score is defined for each cluster by aggregating the geNomad plasmid contig scores with a contig length weighted mean, defining plasmid candidates when cluster scores are larger than the defined threshold. When the geNomad plasmid threshold is larger than 0.5, a fixed geNomad plasmid threshold of 0.5 is applied to the circular contigs, accounting for the circular evidence relatable to plasmids. The nonplasmid clustering strategy consists of two steps. First, the VAMB-latent space is clustered using the iterative medoid clustering algorithm from VAMB. Second, contigs belonging to any plasmid candidate cluster as defined by community-based clustering are removed. In summary, candidate plasmids are first identified using community-based clustering combined with geNomad cluster scoring. Contigs belonging to these plasmid bins are then removed from the bins generated by the density-based clustering strategy.</p><h3 class="c-article__sub-heading" id="Sec22">Binning benchmarking: CAMI2 reassembled</h3><p>We compared the plasmid and organism binning performance of PlasMAAG version 1.0.0, VAMB version 4.1.3, CONCOCT version 1.1.0, MetaBAT2 version 2.12.1, SemiBin2 version 2.1.0, Comebin version 1.0.4, MetaDecoder version 1.0.19, SCAPP version 0.1.4 and MetaPlasmidSPAdes version 3.15.3 over the resimulated CAMI2 datasets. Binning performance was evaluated in terms of the total number of genomes recovered across all samples with precision ≥95% and recall ≥90% (so-called NC genomes). As PlasMAAG, VAMB, MetaBAT2, SemiBin2, Comebin and MetaDecoder perform the binning after assembling the contigs, precision and recall of the bins were obtained from the contig references using BinBencher (version 0.3.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Nissen, J. N., Lindéz, P. P. &amp; Rasmussen, S. BinBencher: fast, flexible and meaningful benchmarking suite for metagenomic binning. Preprint at bioRxiv &#xA;                https://doi.org/10.1101/2024.05.06.592671&#xA;                &#xA;               (2024)." href="/articles/s41587-026-03005-7#ref-CR51" id="ref-link-section-d164666590e2460">51</a></sup>. On the other hand, SCAPP and MetaPlasmidSPAdes assemble their own contigs. Here, we produced a ground truth by aligning the output bins to the origin genomes using NCBI BLAST 2.15.0 accepting hits with an identity &gt;97% and a query coverage &gt; 90%, after which we benchmarked using BinBencher.</p><h3 class="c-article__sub-heading" id="Sec23">Sample benchmarking: CAMI2 reassembled</h3><p>Plasmid purity rate (PPR), plasmid recovery rate (PRR) and <i>F</i><sub>1</sub> were computed for each set of plasmid candidates, reflecting the plasmid characterization at the sample level, not at the bin level. PPR was calculated as the fraction of candidate plasmids reconstructed above precision and recall thresholds in the total number of candidate plasmids output by the method. PRR was calculated as the fraction of candidate plasmids reconstructed above precision and recall thresholds divided by the total number of plasmids with coverage above the recall threshold. Given a sample (<i>s</i>), a set of plasmid candidates (candidates), binning precision and binning recall thresholds (pre, rec) and the set of true plasmids present in the sample (plasmids) are related by the following expressions:</p><div id="Equg" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\begin{array}{l}{\mathrm{PPR}}_{\mathrm{candidates},\mathrm{pre},\mathrm{rec}=\displaystyle \frac{\mathrm{no}.\,\mathrm{of}\,\mathrm{binner}\,\mathrm{plasmid}\,\mathrm{candidates} &gt; (\mathrm{pre}\,,\mathrm{rec})}{\mathrm{no}.\,\mathrm{of}\,\mathrm{binner}\,\mathrm{candidates}}}\end{array}$$</span></div></div><div id="Equh" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\mathrm{PRR}}_{\mathrm{candidates},\,\mathrm{plasmids},\,\mathrm{pre},\,\mathrm{rec}}=\frac{\mathrm{no}.\,\mathrm{of}\,\mathrm{binner}\,\mathrm{plasmid}\,\mathrm{candidates} &gt; (\mathrm{pre},\,\mathrm{rec})}{\mathrm{no}.\,\mathrm{of}\,\mathrm{recoverable}\,\mathrm{plasmids}\,\mathrm{in}\,\mathrm{sample}}$$</span></div></div><p>Lastly, <i>F</i><sub>1</sub> represents the harmonic mean between the PPR and PRR:</p><div id="Equi" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${{F}_{1}}_{\mathrm{PPR},\,\mathrm{PRR}}=\frac{2\times \mathrm{PPR}\times \mathrm{PRR}}{\mathrm{PPR}+\mathrm{PRR}}$$</span></div></div><h3 class="c-article__sub-heading" id="Sec24">Hospital sewage sample sequence datasets</h3><p>Two biological datasets were used in this study to assess the quality of plasmid binning. Urban sewage samples (UWSs) were collected from comparable UWSs from Denmark and Spain located in Odense and Santiago de Compostela, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Yu, Z. et al. Insights into the circular: the cryptic plasmidome and its derived antibiotic resistome in the urban water systems. Environ. Int. 183, 108351 (2024)." href="/articles/s41587-026-03005-7#ref-CR40" id="ref-link-section-d164666590e2773">40</a></sup>. In this study, only hospital sewage samples from each location were used. Sewage samples were collected in the winter and summer of 2018 using ISCO automatic samplers for 24-h flow (50 ml per 5 min) in Denmark and using 24-h time-proportional samples in Spain (mixing hourly samples according to flow information) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">8</a>). Three replicates per site and season were collected on three consecutive days without rain. All samples were initially cooled with ice on site. Then, 100 ml of each sample was centrifuged at 10,000<i>g</i> for 8 min at 4 °C in the laboratory. After removing supernatant, pellets were resuspended in 20% of glycerol stock to reach a final volume of 10 ml for storage at −80 °C. In total, environmental DNA was extracted from all samples using the NucleoSpin soil kit (Macherey-Nagel) using 500 μl of glycerol stock material for direct shotgun metagenomic using Illumina NovaSeq using 2× 150-bp paired-end mode (all 18 samples) and PacBio Sequel2e (five samples from Denmark). PacBio libraries were built from the same DNA extracts using libraries using SMRTbell express template 2.0 kit and Sequel II binding kit 3.2 (Pacific Bioscience) and barcoded using SMRTbell barcoded adaptor plate 3.0 (Pacific Bioscience). Two libraries per 8M SMRTcell (Pacific Bioscience) were pooled and sequenced on a PacBio Sequel2e instrument at the University of Copenhagen.</p><p>For plasmid-enriched samples, we used specific methods to deplete nonplasmid DNA as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Alanin, K. W. S. et al. An improved direct metamobilome approach increases the detection of larger-sized circular elements across kingdoms. Plasmid 115, 102576 (2021)." href="/articles/s41587-026-03005-7#ref-CR52" id="ref-link-section-d164666590e2786">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Kav, A. B. et al. Insights into the bovine rumen plasmidome. Proc. Natl Acad. Sci. USA 109, 5452–5457 (2012)." href="/articles/s41587-026-03005-7#ref-CR53" id="ref-link-section-d164666590e2789">53</a></sup>. Briefly, hospital sewage samples were pretreated by filtration, vortex and sonication and resuspended in TE buffer. Afterward, a prelysis cocktail of cell-wall-degrading enzymes (lysozyme, mutanolysin and lysostaphin) was used to facilitate lysis of Gram-positive bacteria during alkaline lysis. Prelysis was followed by alkaline lysis to remove chromosomal DNA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Green, M. R. &amp; Sambrook, J. Molecular Cloning: A Laboratory Manual 4th edn (Cold Spring Harbor Laboratory, 2012)." href="/articles/s41587-026-03005-7#ref-CR54" id="ref-link-section-d164666590e2793">54</a></sup>, followed by Plasmid-Safe ATP-dependent DNase (Lucigen) digestion. Plasmid-Safe DNase will digest any fragments of dsDNA with open 3′ or 5′ termini, thereby removing fragmented chromosomal DNA. The purified plasmid DNA was then quality-checked and libraries were prepared and sequenced on an Illumina NextSeq platform with a v2.5 sequencing kit (Illumina) in paired-end mode.</p><h3 class="c-article__sub-heading" id="Sec25">Binning benchmarking: hospital sewage</h3><p>We compared the binning performance of PlasMAAG version 1.0.0, VAMB version 4.1.3, MetaBAT2 version 2.12.1, SemiBin2 version 2.1.0, Comebin version 1.0.4, CONCOCT version 1.1.0, MetaDecoder version 1.0.19, metaplasmidSPAdes version 3.15.3 and SCAPP version 0.1.4 on the hospital sewage samples. The dataset consisted of 147,437 short-read contigs assembled from five hospital sewage samples. Each sample was collected at a unique combination of time and location. To evaluate binning performance on these samples, we generated high-quality long-read sequences from the same samples using HiFi sequencing on a Pacific BioSciences Sequel II platform. We assembled the sequences to long-read contigs with hifiasm_meta version 0.3-r073. To determine which short-read contigs corresponded to long-read contigs, we mapped short-read contigs to long-read contigs using minimap2 version 2.24, handling rotated circular contigs by concatenating each long-read contig to itself. We accepted hits with an identity &gt; 97% and a query coverage &gt; 90%. To quantify short-read binning, we counted ‘NC long-read assemblies’ summed across all samples: long-read contigs where a bin of short-read contigs existed such that 90% of the long-read contig was covered with short-read contigs from that bin and 95% of the bin’s base pairs were mapped to no other long-read contig. To evaluate the overall binning performance, the entire set of long-read contigs was used to build the reference, whereas, to evaluate the plasmid binning performance, only the long-read contigs that were circular or with metaplasmidomics reads coverage &gt; 50% were used. Metaplasmidomics short reads were derived from the same samples but underwent a filtering step to enrich for plasmids, thereby representing highly plasmid-enriched sequencing data. NC long-read assemblies were counted with BinBencher version 0.3.0 and NC cellular genomes were estimated with CheckM2 version 0.1.3.</p><h3 class="c-article__sub-heading" id="Sec26">Host–plasmid and intraplasmid diversity exploration</h3><p>PlasMAAG was used to bin the contig sequences from hospital sewage samples from hospitals in Spain. The dataset consisted of <i>n</i> = 828,260 contigs assembled from <i>n</i> = 24 hospital sewage samples. Each sample was collected at a unique combination of time and location. PlasMAAG bins were aggregated into PlasMAAG clusters and classified as plasmids if the aggregated geNomad plasmid score exceeded 0.75. Only plasmids clusters with more than 150 kb were considered for the host–plasmid association. Organism bin quality was estimated with CheckM2 version 0.1.3 and only high-quality (completeness ≥ 70% and precision ≥ 90%) bins were kept. GTDB-tk (version 2.4.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Chaumeil, P.-A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk v2: memory friendly classification with the genome taxonomy database. Bioinformatics 38, 5315–5316 (2022)." href="/articles/s41587-026-03005-7#ref-CR55" id="ref-link-section-d164666590e2819">55</a></sup> was used to estimate taxonomy for the high-quality bins, with cluster taxonomy assigned on the basis of majority vote. Abundance correlation analysis was conducted for plasmids and organism clusters with nonzero abundance over at least 18 of 24 overlapping samples. Spearman correlation coefficients and <i>P</i> values were computed using SciPy’s scipy.stats.spearmanr with default settings, which tests the alternative hypothesis of a nonzero correlation. To account for multiple testing, <i>P</i> values were corrected using the Benjamini–Hochberg, which attempts to correct the false discovery rate, implemented in the statsmodels package. Plasmid cluster hosts were inferred from PLSDB (version 2024_05_21_v2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Molano, L.-A. G., Hirsch, P., Hannig, M., Müller, R. &amp; Keller, A. The PLSDB 2025 update: enhanced annotations and improved functionality for comprehensive plasmid research. Nucleic Acids Res. 53, D189–D196 (2025)." href="/articles/s41587-026-03005-7#ref-CR56" id="ref-link-section-d164666590e2830">56</a></sup> when aligning to any PLSDB entry with &gt;80% identity and &gt;80% coverage. Functional annotations of contigs were performed with anvi’o (version 8)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Eren, A. M. et al. Community-led, integrated, reproducible multi-omics with anvi’o. Nat. Microbiol. 6, 3–6 (2021)." href="/articles/s41587-026-03005-7#ref-CR57" id="ref-link-section-d164666590e2834">57</a></sup> software, using the ‘anvi-run-workflow -w contigs’ command.</p><h3 class="c-article__sub-heading" id="Sec27">Resource usage</h3><p>We evaluated computational resource usage of all methods using the CAMI2 airways reassembled dataset and five samples from the hospital sewage dataset. For the airways dataset, PlasMAAG took 46 min, using 8 threads and 16 GB of RAM. By contrast, SCAPP, excluding the BAM file generation step, took 192 min, using 16 threads and 24 GB of RAM (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">6</a>). Among the other binners, PlasMAAG was slower than VAMB, MetaDecoder and MetaBAT2. For example, VAMB completed the task in just 8 min while using 8 threads and 16 GB of RAM. However, we observed a different trend when evaluating performances on the five hospital sewage samples. When accounting for the additional steps of read assembly and read mapping required to compute abundances, PlasMAAG exhibited similar runtimes to most binners, except for SCAPP, which required much more time. Specifically, PlasMAAG took 3,575 min, VAMB took 3,435 min, ComeBin required 4,911 min and metaplasmidSPAdes took 4,430 min (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">7</a>). By contrast, SCAPP required 116,965 min, 32 times longer than PlasMAAG. This difference in runtime remained consistent even when excluding the read assembly steps (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">8</a>).</p><h3 class="c-article__sub-heading" id="Sec28">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
            
            <p>Reads, contigs and contig annotations for the reassembled CAMI2 datasets are available online (<a href="https://erda.ku.dk/archives/826fe4d8889f88db2ec20058f9eaa015/published-archive.html">https://erda.ku.dk/archives/826fe4d8889f88db2ec20058f9eaa015/published-archive.html</a>, <a href="https://www.erda.dk/archives/fb3dc3896f827891dfd7a0c6f4bf91e1/published-archive.html">https://www.erda.dk/archives/fb3dc3896f827891dfd7a0c6f4bf91e1/published-archive.html</a> and <a href="https://www.erda.dk/archives/18c60aa487d93b4f1d4813c1fd96c0f7/published-archive.html">https://www.erda.dk/archives/18c60aa487d93b4f1d4813c1fd96c0f7/published-archive.html</a>). The metagenomic short reads, metaplasmidomic short reads and metagenomic long reads from the five Danish hospital sewage samples, as well as the metagenomic short reads from the 24 Spanish hospital sewage samples, are available from the European Nucleotide Archive under BioProject <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB85938">PRJEB85938</a>, whereas the assemblies for all samples are available online (<a href="https://erda.ku.dk/archives/e87f0d5e12ca4c1204379d4932c3ae59/published-archive.html">https://erda.ku.dk/archives/e87f0d5e12ca4c1204379d4932c3ae59/published-archive.html</a>; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41587-026-03005-7#MOESM1">9</a>). <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41587-026-03005-7#Sec31">Source data</a> are provided with this paper.</p>
          </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
            
            <p>PlasMAAG is freely available from GitHub (<a href="https://github.com/RasmussenLab/PlasMAAG">https://github.com/RasmussenLab/PlasMAAG</a>), which includes instructions for reproducing main quantitative results, a demo test run, software dependencies and other relevant documentation.</p>
          </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Schwengers, O. et al. Platon: identification and characterization of bacterial plasmid contigs in short-read draft assembliesexploiting protein sequence-based replicon distribution scores. <i>Microb. Genom.</i> <b>6</b>, mgen000398 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32579097" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7660248" aria-label="PubMed Central reference 1">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Platon%3A%20identification%20and%20characterization%20of%20bacterial%20plasmid%20contigs%20in%20short-read%20draft%20assembliesexploiting%20protein%20sequence-based%20replicon%20distribution%20scores&amp;journal=Microb.%20Genom.&amp;volume=6&amp;publication_year=2020&amp;author=Schwengers%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Rodríguez-Beltrán, J., DelaFuente, J., León-Sampedro, R., MacLean, R. C. &amp; San Millán, Á Beyond horizontal gene transfer: the role of plasmids in bacterial evolution. <i>Nat. Rev. Microbiol.</i> <b>19</b>, 347–359 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41579-020-00497-1" data-track-item_id="10.1038/s41579-020-00497-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41579-020-00497-1" aria-label="Article reference 2" data-doi="10.1038/s41579-020-00497-1">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33469168" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Beyond%20horizontal%20gene%20transfer%3A%20the%20role%20of%20plasmids%20in%20bacterial%20evolution&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fs41579-020-00497-1&amp;volume=19&amp;pages=347-359&amp;publication_year=2021&amp;author=Rodr%C3%ADguez-Beltr%C3%A1n%2CJ&amp;author=DelaFuente%2CJ&amp;author=Le%C3%B3n-Sampedro%2CR&amp;author=MacLean%2CRC&amp;author=San%20Mill%C3%A1n%2C%C3%81">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Sherratt, D. J. Bacterial plasmids. <i>Cell</i> <b>3</b>, 189–195 (1974).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/0092-8674(74)90130-5" data-track-item_id="10.1016/0092-8674(74)90130-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2F0092-8674%2874%2990130-5" aria-label="Article reference 3" data-doi="10.1016/0092-8674(74)90130-5">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaE2MXot1KitQ%3D%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4373170" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacterial%20plasmids&amp;journal=Cell&amp;doi=10.1016%2F0092-8674%2874%2990130-5&amp;volume=3&amp;pages=189-195&amp;publication_year=1974&amp;author=Sherratt%2CDJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">San Millan, A. &amp; MacLean, R. C. Fitness costs of plasmids: a limit to plasmid transmission. <i>Microbiol. Spectr.</i> <b>5</b>, 5 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/microbiolspec.MTBP-0016-2017" data-track-item_id="10.1128/microbiolspec.MTBP-0016-2017" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2Fmicrobiolspec.MTBP-0016-2017" aria-label="Article reference 4" data-doi="10.1128/microbiolspec.MTBP-0016-2017">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Fitness%20costs%20of%20plasmids%3A%20a%20limit%20to%20plasmid%20transmission&amp;journal=Microbiol.%20Spectr.&amp;doi=10.1128%2Fmicrobiolspec.MTBP-0016-2017&amp;volume=5&amp;publication_year=2017&amp;author=San%20Millan%2CA&amp;author=MacLean%2CRC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Thomas, C. M. &amp; Nielsen, K. M. Mechanisms of, and barriers to, horizontal gene transfer between bacteria. <i>Nat. Rev. Microbiol.</i> <b>3</b>, 711–721 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrmicro1234" data-track-item_id="10.1038/nrmicro1234" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrmicro1234" aria-label="Article reference 5" data-doi="10.1038/nrmicro1234">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXps1Kjtbk%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16138099" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanisms%20of%2C%20and%20barriers%20to%2C%20horizontal%20gene%20transfer%20between%20bacteria&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fnrmicro1234&amp;volume=3&amp;pages=711-721&amp;publication_year=2005&amp;author=Thomas%2CCM&amp;author=Nielsen%2CKM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Norman, A., Hansen, L. H. &amp; Sørensen, S. J. Conjugative plasmids: vessels of the communal gene pool. <i>Philos. Trans. R. Soc. Lond. B Biol. Sci.</i> <b>364</b>, 2275–2289 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1098/rstb.2009.0037" data-track-item_id="10.1098/rstb.2009.0037" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2009.0037" aria-label="Article reference 6" data-doi="10.1098/rstb.2009.0037">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSmsrzE" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19571247" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2873005" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Conjugative%20plasmids%3A%20vessels%20of%20the%20communal%20gene%20pool&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;doi=10.1098%2Frstb.2009.0037&amp;volume=364&amp;pages=2275-2289&amp;publication_year=2009&amp;author=Norman%2CA&amp;author=Hansen%2CLH&amp;author=S%C3%B8rensen%2CSJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Zhang, T., Zhang, X.-X. &amp; Ye, L. Plasmid metagenome reveals high levels of antibiotic resistance genes and mobile genetic elements in activated sludge. <i>PLoS ONE</i> <b>6</b>, e26041 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0026041" data-track-item_id="10.1371/journal.pone.0026041" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0026041" aria-label="Article reference 7" data-doi="10.1371/journal.pone.0026041">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtl2mtb7F" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22016806" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189950" aria-label="PubMed Central reference 7">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasmid%20metagenome%20reveals%20high%20levels%20of%20antibiotic%20resistance%20genes%20and%20mobile%20genetic%20elements%20in%20activated%20sludge&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0026041&amp;volume=6&amp;publication_year=2011&amp;author=Zhang%2CT&amp;author=Zhang%2CX-X&amp;author=Ye%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Stockdale, S. R. &amp; Hill, C. Incorporating plasmid biology and metagenomics into a holistic model of the human gut microbiome. <i>Curr. Opin. Microbiol.</i> <b>73</b>, 102307 (2023).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.mib.2023.102307" data-track-item_id="10.1016/j.mib.2023.102307" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mib.2023.102307" aria-label="Article reference 8" data-doi="10.1016/j.mib.2023.102307">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmvVWgtbs%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37002975" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Incorporating%20plasmid%20biology%20and%20metagenomics%20into%20a%20holistic%20model%20of%20the%20human%20gut%20microbiome&amp;journal=Curr.%20Opin.%20Microbiol.&amp;doi=10.1016%2Fj.mib.2023.102307&amp;volume=73&amp;publication_year=2023&amp;author=Stockdale%2CSR&amp;author=Hill%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Hugenholtz, P. &amp; Tyson, G. W. Microbiology: metagenomics. <i>Nature</i> <b>455</b>, 481–483 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/455481a" data-track-item_id="10.1038/455481a" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2F455481a" aria-label="Article reference 9" data-doi="10.1038/455481a">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFKhtLnL" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18818648" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbiology%3A%20metagenomics&amp;journal=Nature&amp;doi=10.1038%2F455481a&amp;volume=455&amp;pages=481-483&amp;publication_year=2008&amp;author=Hugenholtz%2CP&amp;author=Tyson%2CGW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Handelsman, J. Metagenomics: application of genomics to uncultured microorganisms. <i>Microbiol. Mol. Biol. Rev.</i> <b>68</b>, 669–685 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/MMBR.68.4.669-685.2004" data-track-item_id="10.1128/MMBR.68.4.669-685.2004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FMMBR.68.4.669-685.2004" aria-label="Article reference 10" data-doi="10.1128/MMBR.68.4.669-685.2004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXjvFSgsQ%3D%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15590779" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC539003" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomics%3A%20application%20of%20genomics%20to%20uncultured%20microorganisms&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;doi=10.1128%2FMMBR.68.4.669-685.2004&amp;volume=68&amp;pages=669-685&amp;publication_year=2004&amp;author=Handelsman%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Yu, M. K., Fogarty, E. C. &amp; Eren, A. M. Diverse plasmid systems and their ecology across human gut metagenomes revealed by PlasX and MobMess. <i>Nat. Microbiol.</i> <b>9</b>, 830–847 (2024).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-024-01610-3" data-track-item_id="10.1038/s41564-024-01610-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-024-01610-3" aria-label="Article reference 11" data-doi="10.1038/s41564-024-01610-3">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXlt1Olu74%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38443576" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10914615" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Diverse%20plasmid%20systems%20and%20their%20ecology%20across%20human%20gut%20metagenomes%20revealed%20by%20PlasX%20and%20MobMess&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-024-01610-3&amp;volume=9&amp;pages=830-847&amp;publication_year=2024&amp;author=Yu%2CMK&amp;author=Fogarty%2CEC&amp;author=Eren%2CAM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Finks, S. S. &amp; Martiny, J. B. H. Plasmid-encoded traits vary across environments. <i>mBio</i> <b>14</b>, e0319122 (2023).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/mbio.03191-22" data-track-item_id="10.1128/mbio.03191-22" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2Fmbio.03191-22" aria-label="Article reference 12" data-doi="10.1128/mbio.03191-22">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36629415" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasmid-encoded%20traits%20vary%20across%20environments&amp;journal=mBio&amp;doi=10.1128%2Fmbio.03191-22&amp;volume=14&amp;publication_year=2023&amp;author=Finks%2CSS&amp;author=Martiny%2CJBH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Pesesky, M. W., Tilley, R. &amp; Beck, D. A. C. Mosaic plasmids are abundant and unevenly distributed across prokaryotic taxa. <i>Plasmid</i> <b>102</b>, 10–18 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.plasmid.2019.02.003" data-track-item_id="10.1016/j.plasmid.2019.02.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.plasmid.2019.02.003" aria-label="Article reference 13" data-doi="10.1016/j.plasmid.2019.02.003">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXjvFGhurw%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30797764" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Mosaic%20plasmids%20are%20abundant%20and%20unevenly%20distributed%20across%20prokaryotic%20taxa&amp;journal=Plasmid&amp;doi=10.1016%2Fj.plasmid.2019.02.003&amp;volume=102&amp;pages=10-18&amp;publication_year=2019&amp;author=Pesesky%2CMW&amp;author=Tilley%2CR&amp;author=Beck%2CDAC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Bouchot, J.-L. et al. in <i>Computational Systems Biology</i> (eds Kriete, A. &amp; Eils, R.) (Elsevier, 2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Rodríguez-Beltrán, J. et al. High recombinant frequency in extraintestinal pathogenic <i>Escherichia</i> <i>coli</i> strains. <i>Mol. Biol. Evol.</i> <b>32</b>, 1708–1716 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/molbev/msv072" data-track-item_id="10.1093/molbev/msv072" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmsv072" aria-label="Article reference 15" data-doi="10.1093/molbev/msv072">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25804522" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20recombinant%20frequency%20in%20extraintestinal%20pathogenic%20Escherichia%20coli%20strains&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsv072&amp;volume=32&amp;pages=1708-1716&amp;publication_year=2015&amp;author=Rodr%C3%ADguez-Beltr%C3%A1n%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Fernandez-Lopez, R., Redondo, S., Garcillan-Barcia, M. P. &amp; de la Cruz, F. Towards a taxonomy of conjugative plasmids. <i>Curr. Opin. Microbiol.</i> <b>38</b>, 106–113 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.mib.2017.05.005" data-track-item_id="10.1016/j.mib.2017.05.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mib.2017.05.005" aria-label="Article reference 16" data-doi="10.1016/j.mib.2017.05.005">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXosl2qsb4%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28586714" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20a%20taxonomy%20of%20conjugative%20plasmids&amp;journal=Curr.%20Opin.%20Microbiol.&amp;doi=10.1016%2Fj.mib.2017.05.005&amp;volume=38&amp;pages=106-113&amp;publication_year=2017&amp;author=Fernandez-Lopez%2CR&amp;author=Redondo%2CS&amp;author=Garcillan-Barcia%2CMP&amp;author=Cruz%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Pellow, D. et al. SCAPP: an algorithm for improved plasmid assembly in metagenomes. <i>Microbiome</i> <b>9</b>, 144 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-021-01068-z" data-track-item_id="10.1186/s40168-021-01068-z" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-021-01068-z" aria-label="Article reference 17" data-doi="10.1186/s40168-021-01068-z">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34172093" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8228940" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=SCAPP%3A%20an%20algorithm%20for%20improved%20plasmid%20assembly%20in%20metagenomes&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-021-01068-z&amp;volume=9&amp;publication_year=2021&amp;author=Pellow%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Antipov, D., Raiko, M., Lapidus, A. &amp; Pevzner, P. A. Plasmid detection and assembly in genomic and metagenomic data sets. <i>Genome Res.</i> <b>29</b>, 961–968 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.241299.118" data-track-item_id="10.1101/gr.241299.118" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.241299.118" aria-label="Article reference 18" data-doi="10.1101/gr.241299.118">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1GrsrrM" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31048319" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6581055" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasmid%20detection%20and%20assembly%20in%20genomic%20and%20metagenomic%20data%20sets&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.241299.118&amp;volume=29&amp;pages=961-968&amp;publication_year=2019&amp;author=Antipov%2CD&amp;author=Raiko%2CM&amp;author=Lapidus%2CA&amp;author=Pevzner%2CPA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Rozov, R. et al. Recycler: an algorithm for detecting plasmids from de novo assembly graphs. <i>Bioinformatics</i> <b>33</b>, 475–482 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btw651" data-track-item_id="10.1093/bioinformatics/btw651" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtw651" aria-label="Article reference 19" data-doi="10.1093/bioinformatics/btw651">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFSgtrfJ" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28003256" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Recycler%3A%20an%20algorithm%20for%20detecting%20plasmids%20from%20de%20novo%20assembly%20graphs&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtw651&amp;volume=33&amp;pages=475-482&amp;publication_year=2017&amp;author=Rozov%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Wajid, B. &amp; Serpedin, E. Review of general algorithmic features for genome assemblers for next generation sequencers. <i>Genomics Proteomics Bioinformatics</i> <b>10</b>, 58–73 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.gpb.2012.05.006" data-track-item_id="10.1016/j.gpb.2012.05.006" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.gpb.2012.05.006" aria-label="Article reference 20" data-doi="10.1016/j.gpb.2012.05.006">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22768980" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5054208" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Review%20of%20general%20algorithmic%20features%20for%20genome%20assemblers%20for%20next%20generation%20sequencers&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;doi=10.1016%2Fj.gpb.2012.05.006&amp;volume=10&amp;pages=58-73&amp;publication_year=2012&amp;author=Wajid%2CB&amp;author=Serpedin%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Ayling, M., Clark, M. D. &amp; Leggett, R. M. New approaches for metagenome assembly with short reads. <i>Brief. Bioinform.</i> <b>21</b>, 584–594 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bib/bbz020" data-track-item_id="10.1093/bib/bbz020" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbib%2Fbbz020" aria-label="Article reference 21" data-doi="10.1093/bib/bbz020">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2ntr7K" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30815668" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20approaches%20for%20metagenome%20assembly%20with%20short%20reads&amp;journal=Brief.%20Bioinform.&amp;doi=10.1093%2Fbib%2Fbbz020&amp;volume=21&amp;pages=584-594&amp;publication_year=2020&amp;author=Ayling%2CM&amp;author=Clark%2CMD&amp;author=Leggett%2CRM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Lander, E. S. &amp; Waterman, M. S. Genomic mapping by fingerprinting random clones: a mathematical analysis. <i>Genomics</i> <b>2</b>, 231–239 (1988).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/0888-7543(88)90007-9" data-track-item_id="10.1016/0888-7543(88)90007-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2F0888-7543%2888%2990007-9" aria-label="Article reference 22" data-doi="10.1016/0888-7543(88)90007-9">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaL1cXlsVeqs7s%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3294162" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20mapping%20by%20fingerprinting%20random%20clones%3A%20a%20mathematical%20analysis&amp;journal=Genomics&amp;doi=10.1016%2F0888-7543%2888%2990007-9&amp;volume=2&amp;pages=231-239&amp;publication_year=1988&amp;author=Lander%2CES&amp;author=Waterman%2CMS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Quince, C. et al. STRONG: metagenomics strain resolution on assembly graphs. <i>Genome Biol.</i> <b>22</b>, 214 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-021-02419-7" data-track-item_id="10.1186/s13059-021-02419-7" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-021-02419-7" aria-label="Article reference 23" data-doi="10.1186/s13059-021-02419-7">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFKmurfK" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34311761" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8311964" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=STRONG%3A%20metagenomics%20strain%20resolution%20on%20assembly%20graphs&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-021-02419-7&amp;volume=22&amp;publication_year=2021&amp;author=Quince%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Nissen, J. N. et al. Improved metagenome binning and assembly using deep variational autoencoders. <i>Nat. Biotechnol.</i> <b>39</b>, 555–560 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41587-020-00777-4" data-track-item_id="10.1038/s41587-020-00777-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-020-00777-4" aria-label="Article reference 24" data-doi="10.1038/s41587-020-00777-4">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXkslSruw%3D%3D" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33398153" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20metagenome%20binning%20and%20assembly%20using%20deep%20variational%20autoencoders&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-00777-4&amp;volume=39&amp;pages=555-560&amp;publication_year=2021&amp;author=Nissen%2CJN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. <i>PeerJ</i> <b>7</b>, e7359 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7717/peerj.7359" data-track-item_id="10.7717/peerj.7359" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7717%2Fpeerj.7359" aria-label="Article reference 25" data-doi="10.7717/peerj.7359">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31388474" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6662567" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=MetaBAT%202%3A%20an%20adaptive%20binning%20algorithm%20for%20robust%20and%20efficient%20genome%20reconstruction%20from%20metagenome%20assemblies&amp;journal=PeerJ&amp;doi=10.7717%2Fpeerj.7359&amp;volume=7&amp;publication_year=2019&amp;author=Kang%2CDD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Wu, Y.-W., Simmons, B. A. &amp; Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. <i>Bioinformatics</i> <b>32</b>, 605–607 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btv638" data-track-item_id="10.1093/bioinformatics/btv638" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv638" aria-label="Article reference 26" data-doi="10.1093/bioinformatics/btv638">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWhur3F" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26515820" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=MaxBin%202.0%3A%20an%20automated%20binning%20algorithm%20to%20recover%20genomes%20from%20multiple%20metagenomic%20datasets&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv638&amp;volume=32&amp;pages=605-607&amp;publication_year=2016&amp;author=Wu%2CY-W&amp;author=Simmons%2CBA&amp;author=Singer%2CSW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Pan, S., Zhao, X.-M. &amp; Coelho, L. P. SemiBin2: self-supervised contrastive learning leads to better MAGs for short- and long- read sequencing. <i>Bioinformatics</i> <b>39</b>, i21–i29 (2023).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btad209" data-track-item_id="10.1093/bioinformatics/btad209" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtad209" aria-label="Article reference 27" data-doi="10.1093/bioinformatics/btad209">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37387171" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10311329" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=SemiBin2%3A%20self-supervised%20contrastive%20learning%20leads%20to%20better%20MAGs%20for%20short-%20and%20long-%20read%20sequencing&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtad209&amp;volume=39&amp;pages=i21-i29&amp;publication_year=2023&amp;author=Pan%2CS&amp;author=Zhao%2CX-M&amp;author=Coelho%2CLP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Pan, S., Zhu, C., Zhao, X.-M. &amp; Coelho, L. P. A deep siamese neural network improves metagenome-assembled genomes in microbiome datasets across different environments. <i>Nat. Commun.</i> <b>13</b>, 2326 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-022-29843-y" data-track-item_id="10.1038/s41467-022-29843-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-022-29843-y" aria-label="Article reference 28" data-doi="10.1038/s41467-022-29843-y">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFOhs73J" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35484115" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9051138" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20deep%20siamese%20neural%20network%20improves%20metagenome-assembled%20genomes%20in%20microbiome%20datasets%20across%20different%20environments&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-29843-y&amp;volume=13&amp;publication_year=2022&amp;author=Pan%2CS&amp;author=Zhu%2CC&amp;author=Zhao%2CX-M&amp;author=Coelho%2CLP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Líndez, P. P. et al. Adversarial and variational autoencoders improve metagenomic binning. <i>Commun. Biol.</i> <b>6</b>, 1073 (2023).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s42003-023-05452-3" data-track-item_id="10.1038/s42003-023-05452-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs42003-023-05452-3" aria-label="Article reference 29" data-doi="10.1038/s42003-023-05452-3">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37865678" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10590447" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Adversarial%20and%20variational%20autoencoders%20improve%20metagenomic%20binning&amp;journal=Commun.%20Biol.&amp;doi=10.1038%2Fs42003-023-05452-3&amp;volume=6&amp;publication_year=2023&amp;author=L%C3%ADndez%2CPP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Lamurias, A., Sereika, M., Albertsen, M., Hose, K. &amp; Nielsen, T. D. Metagenomic binning with assembly graph embeddings. <i>Bioinformatics</i> <b>38</b>, 4481–4487 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btac557" data-track-item_id="10.1093/bioinformatics/btac557" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtac557" aria-label="Article reference 30" data-doi="10.1093/bioinformatics/btac557">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXnt1yit74%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35972375" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9525014" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomic%20binning%20with%20assembly%20graph%20embeddings&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtac557&amp;volume=38&amp;pages=4481-4487&amp;publication_year=2022&amp;author=Lamurias%2CA&amp;author=Sereika%2CM&amp;author=Albertsen%2CM&amp;author=Hose%2CK&amp;author=Nielsen%2CTD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Kutuzova, S. et al. Binning meets taxonomy: TaxVAMB improves metagenome binning using bi-modal variational autoencoder. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2024.10.25.620172" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2024.10.25.620172">https://doi.org/10.1101/2024.10.25.620172</a> (2024).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Johansen, J. et al. Genome binning of viral entities from bulk metagenomics data. <i>Nat. Commun.</i> <b>13</b>, 965 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-022-28581-5" data-track-item_id="10.1038/s41467-022-28581-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-022-28581-5" aria-label="Article reference 32" data-doi="10.1038/s41467-022-28581-5">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xkt1emu7k%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35181661" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8857322" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20binning%20of%20viral%20entities%20from%20bulk%20metagenomics%20data&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-28581-5&amp;volume=13&amp;publication_year=2022&amp;author=Johansen%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Kutuzova, S., Nielsen, M., Piera, P., Nissen, J. N. &amp; Rasmussen, S. Taxometer: improving taxonomic classification of metagenomics contigs. <i>Nat. Commun.</i> <b>15</b>, 8357 (2024).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-024-52771-y" data-track-item_id="10.1038/s41467-024-52771-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-024-52771-y" aria-label="Article reference 33" data-doi="10.1038/s41467-024-52771-y">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=39333501" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11437175" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Taxometer%3A%20improving%20taxonomic%20classification%20of%20metagenomics%20contigs&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-024-52771-y&amp;volume=15&amp;publication_year=2024&amp;author=Kutuzova%2CS&amp;author=Nielsen%2CM&amp;author=Piera%2CP&amp;author=Nissen%2CJN&amp;author=Rasmussen%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Mattock, J. &amp; Watson, M. A comparison of single-coverage and multi-coverage metagenomic binning reveals extensive hidden contamination. <i>Nat. Methods</i> <b>20</b>, 1170–1173 (2023).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-023-01934-8" data-track-item_id="10.1038/s41592-023-01934-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-023-01934-8" aria-label="Article reference 34" data-doi="10.1038/s41592-023-01934-8">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlKjsrrP" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37386187" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20comparison%20of%20single-coverage%20and%20multi-coverage%20metagenomic%20binning%20reveals%20extensive%20hidden%20contamination&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-023-01934-8&amp;volume=20&amp;pages=1170-1173&amp;publication_year=2023&amp;author=Mattock%2CJ&amp;author=Watson%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Abraham, L. louisabraham/fastnode2vec version-0.0.5 <i>Zenodo</i> <a href="https://doi.org/10.5281/zenodo.3902632" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/zenodo.3902632">https://doi.org/10.5281/zenodo.3902632</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Camargo, A. P. et al. Identification of mobile genetic elements with geNomad. <i>Nat. Biotechnol.</i> <b>42</b>, 1303–1312 (2024).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41587-023-01953-y" data-track-item_id="10.1038/s41587-023-01953-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-023-01953-y" aria-label="Article reference 36" data-doi="10.1038/s41587-023-01953-y">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFOltbnM" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37735266" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20mobile%20genetic%20elements%20with%20geNomad&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-023-01953-y&amp;volume=42&amp;pages=1303-1312&amp;publication_year=2024&amp;author=Camargo%2CAP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Sczyrba, A. et al. Critical Assessment of Metagenome Interpretation—a benchmark of metagenomics software. <i>Nat. Methods</i> <b>14</b>, 1063–1071 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.4458" data-track-item_id="10.1038/nmeth.4458" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.4458" aria-label="Article reference 37" data-doi="10.1038/nmeth.4458">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFylu77E" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28967888" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903868" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Critical%20Assessment%20of%20Metagenome%20Interpretation%E2%80%94a%20benchmark%20of%20metagenomics%20software&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4458&amp;volume=14&amp;pages=1063-1071&amp;publication_year=2017&amp;author=Sczyrba%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Mallawaarachchi, V., Wickramarachchi, A. &amp; Lin, Y. GraphBin: refined binning of metagenomic contigs using assembly graphs. <i>Bioinformatics</i> <b>36</b>, 3307–3313 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btaa180" data-track-item_id="10.1093/bioinformatics/btaa180" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtaa180" aria-label="Article reference 38" data-doi="10.1093/bioinformatics/btaa180">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFCktrzM" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32167528" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=GraphBin%3A%20refined%20binning%20of%20metagenomic%20contigs%20using%20assembly%20graphs&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtaa180&amp;volume=36&amp;pages=3307-3313&amp;publication_year=2020&amp;author=Mallawaarachchi%2CV&amp;author=Wickramarachchi%2CA&amp;author=Lin%2CY">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Mallawaarachchi, V. et al. Solving genomic puzzles: computational methods for metagenomic binning. <i>Brief. Bioinform.</i> <b>25</b>, bbae372 (2024).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bib/bbae372" data-track-item_id="10.1093/bib/bbae372" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbib%2Fbbae372" aria-label="Article reference 39" data-doi="10.1093/bib/bbae372">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2MXht1yntLk%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=39082646" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11289683" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Solving%20genomic%20puzzles%3A%20computational%20methods%20for%20metagenomic%20binning&amp;journal=Brief.%20Bioinform.&amp;doi=10.1093%2Fbib%2Fbbae372&amp;volume=25&amp;publication_year=2024&amp;author=Mallawaarachchi%2CV">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Yu, Z. et al. Insights into the circular: the cryptic plasmidome and its derived antibiotic resistome in the urban water systems. <i>Environ. Int.</i> <b>183</b>, 108351 (2024).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.envint.2023.108351" data-track-item_id="10.1016/j.envint.2023.108351" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.envint.2023.108351" aria-label="Article reference 40" data-doi="10.1016/j.envint.2023.108351">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38041983" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Insights%20into%20the%20circular%3A%20the%20cryptic%20plasmidome%20and%20its%20derived%20antibiotic%20resistome%20in%20the%20urban%20water%20systems&amp;journal=Environ.%20Int.&amp;doi=10.1016%2Fj.envint.2023.108351&amp;volume=183&amp;publication_year=2024&amp;author=Yu%2CZ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Tanaka, K. H. et al. The mosaic architecture of <i>Aeromonas</i> <i>salmonicida</i> subsp. <i>salmonicida</i> pAsa4 plasmid and its consequences on antibiotic resistance. <i>PeerJ</i> <b>4</b>, e2595 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7717/peerj.2595" data-track-item_id="10.7717/peerj.2595" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7717%2Fpeerj.2595" aria-label="Article reference 41" data-doi="10.7717/peerj.2595">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27812409" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5088629" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mosaic%20architecture%20of%20Aeromonas%20salmonicida%20subsp.%20salmonicida%20pAsa4%20plasmid%20and%20its%20consequences%20on%20antibiotic%20resistance&amp;journal=PeerJ&amp;doi=10.7717%2Fpeerj.2595&amp;volume=4&amp;publication_year=2016&amp;author=Tanaka%2CKH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Nurk, S., Meleshko, D., Korobeynikov, A. &amp; Pevzner, P. A. metaSPAdes: a new versatile metagenomic assembler. <i>Genome Res.</i> <b>27</b>, 824–834 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.213959.116" data-track-item_id="10.1101/gr.213959.116" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.213959.116" aria-label="Article reference 42" data-doi="10.1101/gr.213959.116">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrrJ" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28298430" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411777" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=metaSPAdes%3A%20a%20new%20versatile%20metagenomic%20assembler&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.213959.116&amp;volume=27&amp;pages=824-834&amp;publication_year=2017&amp;author=Nurk%2CS&amp;author=Meleshko%2CD&amp;author=Korobeynikov%2CA&amp;author=Pevzner%2CPA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Li, H. Minimap2: pairwise alignment for nucleotide sequences. <i>Bioinformatics</i> <b>34</b>, 3094–3100 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bty191" data-track-item_id="10.1093/bioinformatics/bty191" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbty191" aria-label="Article reference 43" data-doi="10.1093/bioinformatics/bty191">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVamu73J" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29750242" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137996" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Minimap2%3A%20pairwise%20alignment%20for%20nucleotide%20sequences&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbty191&amp;volume=34&amp;pages=3094-3100&amp;publication_year=2018&amp;author=Li%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Li, H. et al. The Sequence Alignment/Map format and SAMtools. <i>Bioinformatics</i> <b>25</b>, 2078–2079 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp352" data-track-item_id="10.1093/bioinformatics/btp352" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp352" aria-label="Article reference 44" data-doi="10.1093/bioinformatics/btp352">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505943" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp352&amp;volume=25&amp;pages=2078-2079&amp;publication_year=2009&amp;author=Li%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. <i>J. Mol. Biol.</i> <b>215</b>, 403–410 (1990).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0022-2836(05)80360-2" data-track-item_id="10.1016/S0022-2836(05)80360-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0022-2836%2805%2980360-2" aria-label="Article reference 45" data-doi="10.1016/S0022-2836(05)80360-2">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK3MXitVGmsA%3D%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2231712" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Basic%20local%20alignment%20search%20tool&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2FS0022-2836%2805%2980360-2&amp;volume=215&amp;pages=403-410&amp;publication_year=1990&amp;author=Altschul%2CSF&amp;author=Gish%2CW&amp;author=Miller%2CW&amp;author=Myers%2CEW&amp;author=Lipman%2CDJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Grover, A. &amp; Leskovec, J. Node2vec: scalable feature learning for networks. Preprint at <a href="https://doi.org/10.48550/arXiv.1607.00653" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.48550/arXiv.1607.00653">https://doi.org/10.48550/arXiv.1607.00653</a> (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Nayfach, S. et al. Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome. <i>Nat. Microbiol.</i> <b>6</b>, 960–970 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-021-00928-6" data-track-item_id="10.1038/s41564-021-00928-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-021-00928-6" aria-label="Article reference 47" data-doi="10.1038/s41564-021-00928-6">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVWhtL3O" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34168315" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8241571" aria-label="PubMed Central reference 47">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomic%20compendium%20of%20189%2C680%20DNA%20viruses%20from%20the%20human%20gut%20microbiome&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-021-00928-6&amp;volume=6&amp;pages=960-970&amp;publication_year=2021&amp;author=Nayfach%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Meyer, F. et al. Critical Assessment of Metagenome Interpretation: the second round of challenges. <i>Nat. Methods</i> <b>19</b>, 429–440 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-022-01431-4" data-track-item_id="10.1038/s41592-022-01431-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-022-01431-4" aria-label="Article reference 48" data-doi="10.1038/s41592-022-01431-4">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVWiu7w%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35396482" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9007738" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Critical%20Assessment%20of%20Metagenome%20Interpretation%3A%20the%20second%20round%20of%20challenges&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-022-01431-4&amp;volume=19&amp;pages=429-440&amp;publication_year=2022&amp;author=Meyer%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Fritz, A. et al. CAMISIM: simulating metagenomes and microbial communities. <i>Microbiome</i> <b>7</b>, 17 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-019-0633-6" data-track-item_id="10.1186/s40168-019-0633-6" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-019-0633-6" aria-label="Article reference 49" data-doi="10.1186/s40168-019-0633-6">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30736849" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6368784" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=CAMISIM%3A%20simulating%20metagenomes%20and%20microbial%20communities&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-019-0633-6&amp;volume=7&amp;publication_year=2019&amp;author=Fritz%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Hagberg, D. &amp; Pieter, J. Exploring network structure, dynamics, function using NetworkX. In <i>Proc. 24th Science in Python Conference</i> (eds Ambade, A. et al.) (SciPy, 2025).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Nissen, J. N., Lindéz, P. P. &amp; Rasmussen, S. BinBencher: fast, flexible and meaningful benchmarking suite for metagenomic binning. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2024.05.06.592671" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2024.05.06.592671">https://doi.org/10.1101/2024.05.06.592671</a> (2024).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Alanin, K. W. S. et al. An improved direct metamobilome approach increases the detection of larger-sized circular elements across kingdoms. <i>Plasmid</i> <b>115</b>, 102576 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.plasmid.2021.102576" data-track-item_id="10.1016/j.plasmid.2021.102576" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.plasmid.2021.102576" aria-label="Article reference 52" data-doi="10.1016/j.plasmid.2021.102576">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsVWjs7g%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33872684" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20improved%20direct%20metamobilome%20approach%20increases%20the%20detection%20of%20larger-sized%20circular%20elements%20across%20kingdoms&amp;journal=Plasmid&amp;doi=10.1016%2Fj.plasmid.2021.102576&amp;volume=115&amp;publication_year=2021&amp;author=Alanin%2CKWS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Kav, A. B. et al. Insights into the bovine rumen plasmidome. <i>Proc. Natl Acad. Sci. USA</i> <b>109</b>, 5452–5457 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.1116410109" data-track-item_id="10.1073/pnas.1116410109" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1116410109" aria-label="Article reference 53" data-doi="10.1073/pnas.1116410109">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XlslOjsbs%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325734" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Insights%20into%20the%20bovine%20rumen%20plasmidome&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1116410109&amp;volume=109&amp;pages=5452-5457&amp;publication_year=2012&amp;author=Kav%2CAB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Green, M. R. &amp; Sambrook, J. <i>Molecular Cloning: A Laboratory Manual</i> 4th edn (Cold Spring Harbor Laboratory, 2012).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Chaumeil, P.-A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk v2: memory friendly classification with the genome taxonomy database. <i>Bioinformatics</i> <b>38</b>, 5315–5316 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btac672" data-track-item_id="10.1093/bioinformatics/btac672" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtac672" aria-label="Article reference 55" data-doi="10.1093/bioinformatics/btac672">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXpvFentrY%3D" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36218463" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9710552" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=GTDB-Tk%20v2%3A%20memory%20friendly%20classification%20with%20the%20genome%20taxonomy%20database&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtac672&amp;volume=38&amp;pages=5315-5316&amp;publication_year=2022&amp;author=Chaumeil%2CP-A&amp;author=Mussig%2CAJ&amp;author=Hugenholtz%2CP&amp;author=Parks%2CDH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Molano, L.-A. G., Hirsch, P., Hannig, M., Müller, R. &amp; Keller, A. The PLSDB 2025 update: enhanced annotations and improved functionality for comprehensive plasmid research. <i>Nucleic Acids Res.</i> <b>53</b>, D189–D196 (2025).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkae1095" data-track-item_id="10.1093/nar/gkae1095" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkae1095" aria-label="Article reference 56" data-doi="10.1093/nar/gkae1095">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB28Xhs1eitLw%3D" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=39565221" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20PLSDB%202025%20update%3A%20enhanced%20annotations%20and%20improved%20functionality%20for%20comprehensive%20plasmid%20research&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkae1095&amp;volume=53&amp;pages=D189-D196&amp;publication_year=2025&amp;author=Molano%2CL-AG&amp;author=Hirsch%2CP&amp;author=Hannig%2CM&amp;author=M%C3%BCller%2CR&amp;author=Keller%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Eren, A. M. et al. Community-led, integrated, reproducible multi-omics with anvi’o. <i>Nat. Microbiol.</i> <b>6</b>, 3–6 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-020-00834-3" data-track-item_id="10.1038/s41564-020-00834-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-020-00834-3" aria-label="Article reference 57" data-doi="10.1038/s41564-020-00834-3">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1KktLjE" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33349678" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8116326" aria-label="PubMed Central reference 57">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Community-led%2C%20integrated%2C%20reproducible%20multi-omics%20with%20anvi%E2%80%99o&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-020-00834-3&amp;volume=6&amp;pages=3-6&amp;publication_year=2021&amp;author=Eren%2CAM">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41587-026-03005-7?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>P.P.L., J.N.A., L.S.D, M.P.A. and S.R. were supported by Novo Nordisk Foundation grant NNF23SA0084103. Furthermore, P.P.L., J.N.A., J.N., I.K., S.J.S. and S.R. were supported by Novo Nordisk Foundation grant NNF20OC0062223. L.J.J. and S.R. were supported by Novo Nordisk Foundation grant NNF14CC0001. M.P.A. was funded by the Novo Nordisk Foundation Bioscience PhD program (NNF22SA0078231). The PacBio sequencing Facility at University of Copenhagen’s Biology Department is supported by the Novo Nordisk Foundation (NNF20OC0061528 to S.J.S.).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark</p><p class="c-article-author-affiliation__authors-list">Pau Piera Líndez, Lasse Schnell Danielsen, Marc Pielies Avellí, Jakob Nybo Andersen &amp; Simon Rasmussen</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Section of Microbiology, Department of Biology, University of Copenhagen, Copenhagen, Denmark</p><p class="c-article-author-affiliation__authors-list">Iva Kovačić, Joseph Nesme &amp; Søren Johannes Sørensen</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Novo Nordisk Foundation Center for Protein Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark</p><p class="c-article-author-affiliation__authors-list">Lars Juhl Jensen</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Pau-Piera_L_ndez-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Pau Piera Líndez</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pau%20Piera%20L%C3%ADndez" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pau%20Piera%20L%C3%ADndez" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pau%20Piera%20L%C3%ADndez%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lasse_Schnell-Danielsen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Lasse Schnell Danielsen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lasse%20Schnell%20Danielsen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lasse%20Schnell%20Danielsen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lasse%20Schnell%20Danielsen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Iva-Kova_i_-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Iva Kovačić</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Iva%20Kova%C4%8Di%C4%87" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Iva%20Kova%C4%8Di%C4%87" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Iva%20Kova%C4%8Di%C4%87%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marc-Pielies_Avell_-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Marc Pielies Avellí</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marc%20Pielies%20Avell%C3%AD" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marc%20Pielies%20Avell%C3%AD" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marc%20Pielies%20Avell%C3%AD%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Joseph-Nesme-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Joseph Nesme</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joseph%20Nesme" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joseph%20Nesme" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joseph%20Nesme%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lars_Juhl-Jensen-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Lars Juhl Jensen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lars%20Juhl%20Jensen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lars%20Juhl%20Jensen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lars%20Juhl%20Jensen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jakob_Nybo-Andersen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Jakob Nybo Andersen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jakob%20Nybo%20Andersen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jakob%20Nybo%20Andersen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jakob%20Nybo%20Andersen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-S_ren_Johannes-S_rensen-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Søren Johannes Sørensen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=S%C3%B8ren%20Johannes%20S%C3%B8rensen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=S%C3%B8ren%20Johannes%20S%C3%B8rensen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22S%C3%B8ren%20Johannes%20S%C3%B8rensen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Simon-Rasmussen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Simon Rasmussen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Simon%20Rasmussen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">Search author on:</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Simon%20Rasmussen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simon%20Rasmussen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>S.R. and S.J.S. conceptualized the study. S.R., J.N.A. and P.P.L. guided the analysis. P.P.L. developed PlasMAAG, wrote the software and performed the analysis. J.N.A. and L.S.D. also wrote the software. J.N.A. also performed analyses. S.J.S., J.N., M.P.A. and L.J.J. provided input for the analysis. I.K. and J.N. generated the sewage data. P.P.L., J.N.A. and S.R. wrote the manuscript with contributions from all coauthors. All authors read and approved the final version of the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" aria-label="email Jakob Nybo Andersen" href="mailto:jakob.nissen@sund.ku.dk">Jakob Nybo Andersen</a>, <a id="corresp-c2" aria-label="email Søren Johannes Sørensen" href="mailto:sjs@bio.ku.dk">Søren Johannes Sørensen</a> or <a id="corresp-c3" aria-label="email Simon Rasmussen" href="mailto:srasmuss@sund.ku.dk">Simon Rasmussen</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
            
              <h3 class="c-article__sub-heading" id="FPar2">Competing interests</h3>
              <p>S.R. is the founder and owner of the Danish company BioAI. S.R has received a research grant and performed consulting for Sidera Bio ApS. The remaining authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
            
            
              <h3 class="c-article__sub-heading" id="FPar1">Peer review information</h3>
              <p><i>Nature Biotechnology</i> thanks Antonio Camargo and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
            
          </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec30-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec30">Supplementary information</h2><div class="c-article-section__content" id="Sec30-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Notes 1–14, Supplementary Figs. 1–30 and Supplementary Tables 1–4 and 6–9.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Table 5</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Table 5. Phage identifiers from the MGV.</p></div></div></div></div></div></section><section data-title="Source data"><div class="c-article-section" id="Sec31-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec31">Source data</h2><div class="c-article-section__content" id="Sec31-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data figs. 2–5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41587-026-03005-7/MediaObjects/41587_2026_3005_MOESM4_ESM.xlsx" data-supp-info-image="">Source Data Figs. 2–5</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Source data required to reproduce all the main figures.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Accurate%20plasmid%20reconstruction%20from%20metagenomics%20data%20using%20assembly%E2%80%93alignment%20graphs%20and%20contrastive%20learning&amp;author=Pau%20Piera%20L%C3%ADndez%20et%20al&amp;contentID=10.1038%2Fs41587-026-03005-7&amp;copyright=The%20Author%28s%29&amp;publication=1087-0156&amp;publicationDate=2026-02-04&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41587-026-03005-7" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41587-026-03005-7" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Piera Líndez, P., Danielsen, L.S., Kovačić, I. <i>et al.</i> Accurate plasmid reconstruction from metagenomics data using assembly–alignment graphs and contrastive learning.
                    <i>Nat Biotechnol</i>  (2026). https://doi.org/10.1038/s41587-026-03005-7</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41587-026-03005-7?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2025-03-27">27 March 2025</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2026-01-07">07 January 2026</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2026-02-04">04 February 2026</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Version of record<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2026-02-04">04 February 2026</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41587-026-03005-7</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy shareable link to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
            

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="u-mb-48 js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41587-026-03005-7.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                    
                
            
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/biotech.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41587-026-03005-7;doi=10.1038/s41587-026-03005-7;subjmeta=114,1305,1386,631;kwrd=Classification+and+taxonomy,Machine+learning">
        
        <script>
            window.SN = window.SN || {};
            window.SN.libs = window.SN.libs || {};
            window.SN.libs.ads = window.SN.libs.ads || {};
            window.SN.libs.ads.slotConfig = window.SN.libs.ads.slotConfig || {};
            
                window.SN.libs.ads.slotConfig['right'] = {
                    'pos': 'right',
                    'type': 'article',
                    'path': 's41587-026-03005-7'
                };
            
            
            window.SN.libs.ads.slotConfig['kwrd'] = 'Classification+and+taxonomy,Machine+learning';
            
            
            window.SN.libs.ads.slotConfig['subjmeta'] = '114,1305,1386,631';
            
            
        </script>
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/biotech.nature.com/article&amp;sz=300x250&amp;c=484371133&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41587-026-03005-7%26doi%3D10.1038/s41587-026-03005-7%26subjmeta%3D114,1305,1386,631%26kwrd%3DClassification+and+taxonomy,Machine+learning">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/biotech.nature.com/article&amp;sz=300x250&amp;c=484371133&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41587-026-03005-7%26doi%3D10.1038/s41587-026-03005-7%26subjmeta%3D114,1305,1386,631%26kwrd%3DClassification+and+taxonomy,Machine+learning"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/podcasts"
                                   data-track="click"
                                   data-track-action="podcasts"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://www.facebook.com/NatBiotechnol"
                               data-track="click"
                               data-track-action="facebook"
                               data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/NatureBiotech"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://journal-alerts.springernature.com/subscribe?journal_id&#x3D;41587"
                               rel="nofollow"
                               data-track="nav_sign_up_for_alerts"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/nbt.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="nav_language_services"
                                   data-track-context="header publish with us dropdown menu"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/nbt/open-access-funding"
                                   data-test="funding-eligibility-link"
                                   data-track="click_explore_funding"
                                   data-track-context="header publish with us"
                                   data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nbt.nature.com/cgi-bin/main.plex"
                                   data-track="click_submit_manuscript"
                                   data-track-context="submit link in Nature header dropdown menu"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external
                                   data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="nbt">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Biotechnology
                    </span>
                    (<i itemprop="alternateName">Nat Biotechnol</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-1696</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1087-0156</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/"
                                                  data-track="click" data-track-action="protocols.io"
                                                  data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/immersive/natureawards/index.html"
                                                  data-track="click" data-track-action="nature awards"
                                                  data-track-label="link">Nature Awards</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2026 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18ZM8.707 7.293 12 10.585l3.293-3.292a1 1 0 0 1 1.414 1.414L13.415 12l3.292 3.293a1 1 0 0 1-1.414 1.414L12 13.415l-3.293 3.292a1 1 0 1 1-1.414-1.414L10.585 12 7.293 8.707a1 1 0 0 1 1.414-1.414Z"/></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"/><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"/><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"/><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"/></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 22 18"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-marker-unfilled"><path d="M17.7998 3.57751C17.7998 3.32977 17.6471 3.17712 17.3994 3.17712H6.5C6.25231 3.17712 6.09961 3.32977 6.09961 3.57751V20.6244L11.4248 16.8871L11.5596 16.807C11.8831 16.6484 12.2726 16.6755 12.5742 16.8871L17.7998 20.5531V3.57751ZM19.7998 21.1049C19.7998 21.7661 19.4529 22.3363 18.9053 22.6263C18.3697 22.9099 17.7222 22.8847 17.1855 22.5629C17.1651 22.5506 17.1445 22.5375 17.125 22.5238L11.999 18.9261L6.87402 22.5238C6.85458 22.5374 6.83481 22.5506 6.81445 22.5629C6.25918 22.896 5.60347 22.8929 5.08398 22.6469C4.54665 22.3923 4.09961 21.8431 4.09961 21.1049V3.57751C4.09961 2.22526 5.14769 1.17712 6.5 1.17712H17.3994C18.7517 1.17712 19.7998 2.22526 19.7998 3.57751V21.1049Z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-orcid-logo" viewBox="0 0 40 40"><path fill-rule="evenodd" d="M12.281 10.453c.875 0 1.578-.719 1.578-1.578 0-.86-.703-1.578-1.578-1.578-.875 0-1.578.703-1.578 1.578 0 .86.703 1.578 1.578 1.578Zm-1.203 18.641h2.406V12.359h-2.406v16.735Z"/><path fill-rule="evenodd" d="M17.016 12.36h6.5c6.187 0 8.906 4.421 8.906 8.374 0 4.297-3.36 8.375-8.875 8.375h-6.531V12.36Zm6.234 14.578h-3.828V14.53h3.703c4.688 0 6.828 2.844 6.828 6.203 0 2.063-1.25 6.203-6.703 6.203Z" clip-rule="evenodd"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41587-026-03005-7&amp;format=js&amp;last_modified=2026-02-04" async></script>
</body>
</html>